# The Effect of Loss of Basement Membrane Collagen on the Nucleus
PhD Thesis <br>
Simranpreet Kaur Summan <br>
PhD Supervisors: Professor Edel O’Toole, Dr Matthew Caley, Professor Julien Gautrot, Dr Andrew Mayes and Dr David Gunn.

# Please click on this private link to view the contents or open the zipped folder to gain access. 
https://drive.google.com/file/d/188Guyo5KLeuc0BA7AEb0mCQIOZ8A6Xwa/view?usp=share_link 

# Note. Download the zipped folder called main_protected.zip. Then click on it. Enter the password. Then open main.zip to view contents. 


# PhD Thesis
# The Effect of Loss of Basement Membrane Collagen on the Nucleus
Simranpreet Kaur Summan <br>
PhD Supervisors: Professor Edel O’Toole (primary), Dr Matthew Caley, Professor Julien Gautrot
PhD Industry Supervisors: Dr Andrew Mayes (Unilever) and Dr David Gunn (Unilever)
Academic Collaborators: Prof. Pradeepa Madapura and Dr Manthan Patel 

## Bibliography (Formatted)
Please see the fully formatted bibliography.

Bibliography

1.	McGrath JA. Structure and function of skin, hair and nails. Medicine. 2025 Jul;53(7):417–22. doi:10.1016/j.mpmed.2025.04.013 
2.	Gallo RL. Human skin is the largest epithelial surface for interaction with microbes. Journal of Investigative Dermatology. 2017 Jun;137(6):1213–4. doi:10.1016/j.jid.2016.11.045 
3.	Abdo JM, Sopko NA, Milner SM. The applied anatomy of Human skin: A model for regeneration. Wound Medicine. 2020 Mar;28:100179. doi:10.1016/j.wndm.2020.100179 
4.	Boer M, Duchnik E, Maleszka R, Marchlewicz M. Structural and biophysical characteristics of human skin in maintaining proper epidermal barrier function. Advances in Dermatology and Allergology. 2016;1:1–5. doi:10.5114/pdia.2015.48037 
5.	Silver FH, Siperko LM, Seehra GP. Mechanobiology of force transduction in dermal tissue. Skin Research and Technology. 2003 Jan 20;9(1):3–23. doi:10.1034/j.1600-0846.2003.00358.x 
6.	Pinheiro D, Bellaϊche Y. Mechanical force-driven adherens junction remodeling and epithelial dynamics. Developmental Cell. 2018 Nov;47(3):391. doi:10.1016/j.devcel.2018.10.021 
7.	Boyle CJ, Plotczyk M, Villalta SF, Patel S, Hettiaratchy S, Masouros SD, et al. Morphology and composition play distinct and complementary roles in the tolerance of plantar skin to mechanical load. Science Advances. 2019 Oct 11;5(10). doi:10.1126/sciadv.aay0244 
8.	Bouten CV, Oomens CW, Baaijens FP, Bader DL. The etiology of pressure ulcers: Skin deep or muscle bound? Archives of Physical Medicine and Rehabilitation. 2003 Apr;84(4):616–9. doi:10.1053/apmr.2003.50038 
9.	LeBlanc K, Baranoski S. Skin tears: State of the science. Advances in Skin &amp; Wound Care. 2011 Sept;24(9):2–15. doi:10.1097/01.asw.0000405316.99011.95 
10.	Chen SX, Zhang L-J, Gallo RL. Dermal white adipose tissue: A newly recognized layer of skin innate defense. Journal of Investigative Dermatology. 2019 May;139(5):1002–9. doi:10.1016/j.jid.2018.12.031 
11.	Watt FM. Mammalian skin cell biology: At the Interface Between Laboratory and Clinic. Science. 2014 Nov 21;346(6212):937–40. doi:10.1126/science.1253734 
12.	Tracy LE, Minasian RA, Caterson EJ. Extracellular matrix and dermal fibroblast function in the healing wound. Advances in Wound Care. 2016 Mar;5(3):119–36. doi:10.1089/wound.2014.0561 
13.	Sirois JP, Heinz A. Matrikines in the skin: Origin, effects, and therapeutic potential. Pharmacology &amp; Therapeutics. 2024 Aug;260:108682. doi:10.1016/j.pharmthera.2024.108682 
14.	Baroni A, Buommino E, De Gregorio V, Ruocco E, Ruocco V, Wolf R. Structure and function of the epidermis related to barrier properties. Clinics in Dermatology. 2012 May;30(3):257–62. doi:10.1016/j.clindermatol.2011.08.007 
15.	Driskell RR, Jahoda CA, Chuong C, Watt FM, Horsley V. Defining dermal adipose tissue. Experimental Dermatology. 2014 Sept;23(9):629–31. doi:10.1111/exd.12450 
16.	Biggs LC, Kim CS, Miroshnikova YA, Wickström SA. Mechanical forces in the skin: Roles in tissue architecture, stability, and function. Journal of Investigative Dermatology. 2020 Feb;140(2):284–90. doi:10.1016/j.jid.2019.06.137 
17.	Sachs D, Wahlsten A, Kozerke S, Restivo G, Mazza E. A biphasic multilayer computational model of Human skin. Biomechanics and Modeling in Mechanobiology. 2021 Feb 10;20(3):969–82. doi:10.1007/s10237-021-01424-w 
18.	Briggaman RA, Wheeler CE. The epidermal-dermal junction. Journal of Investigative Dermatology. 1975 Jul;65(1):71–84. doi:10.1111/1523-1747.ep12598050 
19.	Candiello J, Balasubramani M, Schreiber EM, Cole GJ, Mayer U, Halfter W, et al. Biomechanical properties of native basement membranes. The FEBS Journal. 2007 May 4;274(11):2897–908. doi:10.1111/j.1742-4658.2007.05823.x 
20.	Henrich PB, Monnier CA, Halfter W, Haritoglou C, Strauss RW, Lim RY, et al. Nanoscale topographic and biomechanical studies of the human internal limiting membrane. Investigative Opthalmology &amp; Visual Science. 2012 Jun 28;53(6):2561. doi:10.1167/iovs.11-8502 
21.	Boyer G, Laquièze L, Le Bot A, Laquièze S, Zahouani H. Dynamic indentation on human skin in vivo: Ageing effects. Skin Research and Technology. 2009 Jan 15;15(1):55–67. doi:10.1111/j.1600-0846.2008.00324.x 
22.	Alexander H, Cook T. Variations with age in the mechanical properties of human skin in vivo. Journal of Tissue Viability. 2006 Aug;16(3):6–11. doi:10.1016/s0965-206x(06)63002-7 
23.	Agache PG, Monneur C, Leveque JL, De Rigal J. Mechanical properties and Young’s modulus of human skin in vivo. Archives of Dermatological Research. 1980 Dec;269(3):221–32. doi:10.1007/bf00406415 
24.	Diridollou S, Patat F, Gens F, Vaillant L, Black D, Lagarde JM, et al. In vivo model of the mechanical properties of the human skin under suction. Skin Research and Technology. 2000 Nov;6(4):214–21. doi:10.1034/j.1600-0846.2000.006004214.x 
25.	Ní Annaidh A, Bruyère K, Destrade M, Gilchrist MD, Maurini C, Otténio M, et al. Automated estimation of collagen fibre dispersion in the dermis and its contribution to the anisotropic behaviour of skin. Annals of Biomedical Engineering. 2012 Mar 17;40(8):1666–78. doi:10.1007/s10439-012-0542-3 
26.	Meador WD, Sugerman GP, Story HM, Seifert AW, Bersi MR, Tepole AB, et al. The regional-dependent biaxial behavior of young and aged Mouse skin: A detailed histomechanical characterization, residual strain analysis, and Constitutive Model. Acta Biomaterialia. 2020 Jan;101:403–13. doi:10.1016/j.actbio.2019.10.020 
27.	Blanpain C, Fuchs E. Epidermal homeostasis: A balancing act of stem cells in the skin. Nature Reviews Molecular Cell Biology. 2009 Feb 11;10(3):207–17. doi:10.1038/nrm2636 
28.	Lechler T, Fuchs E. Asymmetric cell divisions promote stratification and differentiation of mammalian skin. Nature. 2005 Aug 10;437(7056):275–80. doi:10.1038/nature03922 
29.	Kowalczyk AP, Green KJ. Structure, function, and regulation of Desmosomes. Progress in Molecular Biology and Translational Science. 2013;95–118. doi:10.1016/b978-0-12-394311-8.00005-4 
30.	Bragulla HH, Homberger DG. Structure and functions of keratin proteins in simple, stratified, keratinized and cornified epithelia. Journal of Anatomy. 2009 Apr;214(4):516–59. doi:10.1111/j.1469-7580.2009.01066.x 
31.	Rogerson C, Bergamaschi D, O’Shaughnessy RF. Uncovering mechanisms of nuclear degradation in keratinocytes: A paradigm for nuclear degradation in other tissues. Nucleus. 2018 Jan 3;9(1):56–64. doi:10.1080/19491034.2017.1412027 
32.	Balda MS, Matter K. Tight junctions and the regulation of gene expression. Biochimica et Biophysica Acta (BBA) - Biomembranes. 2009 Apr;1788(4):761–7. doi:10.1016/j.bbamem.2008.11.024 
33.	Rübsam M, Mertz AF, Kubo A, Marg S, Jüngst C, Goranci-Buzhala G, et al. E-cadherin integrates mechanotransduction and EGFR signaling to control junctional tissue polarization and tight junction positioning. Nature Communications. 2017 Nov 1;8(1). doi:10.1038/s41467-017-01170-7 
34.	Furue M. Regulation of filaggrin, loricrin, and involucrin by il-4, IL-13, IL-17A, IL-22, ahr, and Nrf2: Pathogenic implications in atopic dermatitis. International Journal of Molecular Sciences. 2020 Jul 29;21(15):5382. doi:10.3390/ijms21155382 
35.	Lee SE, Lee SH. Skin barrier and calcium. Annals of Dermatology. 2018;30(3):265. doi:10.5021/ad.2018.30.3.265 
36.	Marenholz I, Zirra M, Fischer DF, Backendorf C, Ziegler A, Mischke D. Identification of human epidermal differentiation complex (edc)-encoded genes by subtractive hybridization of entire yacs to a gridded keratinocyte cdna library. Genome Research. 2001 Mar 1;11(3):341–55. doi:10.1101/gr.114801 
37.	Biggs LC, Kim CS, Miroshnikova YA, Wickström SA. Mechanical forces in the skin: Roles in tissue architecture, stability, and function. Journal of Investigative Dermatology. 2020 Feb;140(2):284–90. doi:10.1016/j.jid.2019.06.137 
38.	Miroshnikova YA, Le HQ, Schneider D, Thalheim T, Rübsam M, Bremicker N, et al. Adhesion forces and cortical tension couple cell proliferation and differentiation to drive epidermal stratification. Nature Cell Biology. 2017 Dec 11;20(1):69–80. doi:10.1038/s41556-017-0005-z 
39.	Kypriotou M, Huber M, Hohl D. The human epidermal differentiation complex: Cornified envelope precursors, S100 proteins and the ‘fused genes’ family. Experimental Dermatology. 2012 Apr 16;21(9):643–9. doi:10.1111/j.1600-0625.2012.01472.x 
40.	Głowacka A, Bieganowski P, Jurewicz E, Leśniak W, Wilanowski T, Filipek A. Regulation of s100a10 gene expression. Biomolecules. 2021 Jul 2;11(7):974. doi:10.3390/biom11070974 
41.	Belyi VA, Ak P, Markert E, Wang H, Hu W, Puzio-Kuter A, et al. The origins and evolution of the p53 family of genes. Cold Spring Harbor Perspectives in Biology. 2010 Dec 16;2(6). doi:10.1101/cshperspect.a001198 
42.	Fessing MY, Mardaryev AN, Gdula MR, Sharov AA, Sharova TY, Rapisarda V, et al. P63 regulates satb1 to control tissue-specific chromatin remodeling during development of the epidermis. Journal of Cell Biology. 2011 Sept 19;194(6):825–39. doi:10.1083/jcb.201101148 
43.	Fessing MY, Mardaryev AN, Gdula MR, Sharov AA, Sharova TY, Rapisarda V, et al. P63 regulates satb1 to control tissue-specific chromatin remodeling during development of the epidermis. Journal of Cell Biology. 2011 Sept 19;194(6):825–39. doi:10.1083/jcb.201101148 
44.	Gdula MR, Poterlowicz K, Mardaryev AN, Sharov AA, Peng Y, Fessing MY, et al. Remodeling of three-dimensional organization of the nucleus during terminal keratinocyte differentiation in the epidermis. Journal of Investigative Dermatology. 2013 Sept;133(9):2191–201. doi:10.1038/jid.2013.66 
45.	Botchkarev VA, Fessing MY, Sharov AA. Deciphering a message from the nucleus: How transcription factors and spatial chromatin interactions orchestrate epidermal differentiation. Journal of Investigative Dermatology. 2023 Jul;143(7):1117–20. doi:10.1016/j.jid.2023.02.013 
46.	Dainichi T, Nishie W, Yamagami Y, Sonobe H, Ujiie H, Kaku Y, et al. Bullous pemphigoid suggestive of complement‐independent Blister Formation with anti‐                     bp                     180 igg4 autoantibodies. British Journal of Dermatology. 2016 Mar 18;175(1):187–90. doi:10.1111/bjd.14411 
47.	Farage MA, Miller KW, Elsner P, Maibach HI. Characteristics of the aging skin. Advances in Wound Care. 2013 Feb;2(1):5–10. doi:10.1089/wound.2011.0356 
48.	Orioli D, Dellambra E. Epigenetic regulation of skin cells in natural aging and premature aging diseases. Cells. 2018 Dec 12;7(12):268. doi:10.3390/cells7120268 
49.	Zhang S, Duan E. Fighting against skin aging. Cell Transplantation. 2018 Apr 25;27(5):729–38. doi:10.1177/0963689717725755 
50.	Roig-Rosello E, Rousselle P. The human epidermal basement membrane: A shaped and cell instructive platform that aging slowly alters. Biomolecules. 2020 Nov 27;10(12):1607. doi:10.3390/biom10121607 
51.	Karim PL, Inda Astri Aryani, Nopriyati. Anatomy and histologic of Intrinsic aging skin. Bioscientia Medicina : Journal of Biomedicine and Translational Research. 2021 Oct 25;5(11):1165–77. doi:10.32539/bsm.v5i11.417 
52.	Langton AK, Halai P, Griffiths CEM, Sherratt MJ, Watson REB. The impact of intrinsic ageing on the protein composition of the dermal-epidermal junction. Mechanisms of Ageing and Development. 2016 Jun;156:14–6. doi:10.1016/j.mad.2016.03.006 
53.	Hayflick L, Moorhead PS. The serial cultivation of human diploid cell strains. Experimental Cell Research. 1961 Dec;25(3):585–621. doi:10.1016/0014-4827(61)90192-6 
54.	Chin T, Lee XE, Ng PY, Lee Y, Dreesen O. The role of cellular senescence in skin aging and age-related skin pathologies. Frontiers in Physiology. 2023 Nov 22;14. doi:10.3389/fphys.2023.1297637 
55.	Wang AS, Dreesen O. Biomarkers of cellular senescence and skin aging. Frontiers in Genetics. 2018 Aug 23;9. doi:10.3389/fgene.2018.00247 
56.	Rodier F, Campisi J. Four faces of cellular senescence. Journal of Cell Biology. 2011 Feb 14;192(4):547–56. doi:10.1083/jcb.201009094 
57.	Zhang R, Chen W, Adams PD. Molecular dissection of formation of senescence-associated heterochromatin foci. Molecular and Cellular Biology. 2007 Mar 1;27(6):2343–58. doi:10.1128/mcb.02019-06 
58.	Dimri GP, Lee X, Basile G, Acosta M, Scott G, Roskelley C, et al. A biomarker that identifies senescent human cells in culture and in aging skin in vivo. Proceedings of the National Academy of Sciences. 1995 Sept 26;92(20):9363–7. doi:10.1073/pnas.92.20.9363 
59.	Dreesen O, Chojnowski A, Ong PF, Zhao TY, Common JE, Lunny D, et al. Lamin B1 fluctuations have differential effects on cellular proliferation and senescence. Journal of Cell Biology. 2013 Feb 25;200(5):605–17. doi:10.1083/jcb.201206121 
60.	Low E, Alimohammadiha G, Smith LA, Costello LF, Przyborski SA, von Zglinicki T, et al. How good is the evidence that cellular senescence causes skin ageing? Ageing Research Reviews. 2021 Nov;71:101456. doi:10.1016/j.arr.2021.101456 
61.	Dorf N, Maciejczyk M. Skin senescence—from basic research to clinical practice. Frontiers in Medicine. 2024 Oct 18;11. doi:10.3389/fmed.2024.1484345 
62.	López-Otín C, Blasco MA, Partridge L, Serrano M, Kroemer G. The hallmarks of aging. Cell. 2013 Jun;153(6):1194–217. doi:10.1016/j.cell.2013.05.039 
63.	López-Otín C, Blasco MA, Partridge L, Serrano M, Kroemer G. Hallmarks of aging: An expanding universe. Cell. 2023 Jan;186(2):243–78. doi:10.1016/j.cell.2022.11.001 
64.	en P, Shah PP, Nativio R, Berger SL. Epigenetic mechanisms of longevity and aging. Cell. 2016 Aug;166(4):822–39. doi:10.1016/j.cell.2016.07.050 
65.	Aleemardani M, Trikić MZ, Green NH, Claeyssens F. The importance of mimicking dermal-epidermal junction for Skin Tissue Engineering: A Review. Bioengineering. 2021 Oct 20;8(11):148. doi:10.3390/bioengineering8110148 
66.	Goletz S, Zillikens D, Schmidt E. Structural proteins of the dermal‐epidermal junction targeted by autoantibodies in pemphigoid diseases. Experimental Dermatology. 2017 Dec;26(12):1154–62. doi:10.1111/exd.13446 
67.	Ramms L, Fabris G, Windoffer R, Schwarz N, Springer R, Zhou C, et al. Keratins as the main component for the mechanical integrity of keratinocytes. Proceedings of the National Academy of Sciences. 2013 Oct 28;110(46):18513–8. doi:10.1073/pnas.1313491110 
68.	Karim PL, Inda Astri Aryani, Nopriyati. Anatomy and histologic of Intrinsic aging skin. Bioscientia Medicina : Journal of Biomedicine and Translational Research. 2021 Oct 25;5(11):1165–77. doi:10.32539/bsm.v5i11.417 
69.	Sánchez-Jimeno C, Escámez MJ, Ayuso C, Trujillo-Tiebas MJ, del Río M. Genetic diagnosis of epidermolysis bullosa: Recommendations from an expert Spanish Research Group. Actas Dermo-Sifiliográficas (English Edition). 2018 Mar;109(2):104–22. doi:10.1016/j.adengl.2017.12.005 
70.	Hegde S, Raghavan S. A skin-depth analysis of integrins: Role of the integrin network in health and disease. Cell Communication &amp; Adhesion. 2013 Nov 8;20(6):155–69. doi:10.3109/15419061.2013.854334 
71.	Kleiser S, Nyström A. Interplay between cell-surface receptors and extracellular matrix in skin. Biomolecules. 2020 Aug 11;10(8):1170. doi:10.3390/biom10081170 
72.	Ludwig BS, Kessler H, Kossatz S, Reuning U. RGD-binding integrins revisited: How recently discovered functions and novel synthetic ligands (re-)shape an ever-evolving field. Cancers. 2021 Apr 4;13(7):1711. doi:10.3390/cancers13071711 
73.	 Woodley DT, Atha T, Huang Y, Chen M, Krueger GG, Jorgensen CM, et al. Normal and gene-corrected dystrophic epidermolysis bullosa fibroblasts alone can produce type VII collagen at the basement membrane zone. Journal of Investigative Dermatology. 2003 Nov;121(5):1021–8. doi:10.1046/j.1523-1747.2003.12571.x 
74.	Chen M, Marinkovich MP, Veis A, Cai X, Rao CN, O’Toole EA, et al. Interactions of the amino-terminal noncollagenous (NC1) domain of type VII collagen with extracellular matrix components. Journal of Biological Chemistry. 1997 Jun;272(23):14516–22. doi:10.1074/jbc.272.23.14516 
75.	Van Agtmael T, Bruckner-Tuderman L. Basement membranes and human disease. Cell and Tissue Research. 2009 Sept 15;339(1):167–88. doi:10.1007/s00441-009-0866-y 
76.	Bruckner-Tuderman L. Can type VII collagen injections cure dystrophic epidermolysis bullosa? Molecular Therapy. 2009 Jan;17(1):6–7. doi:10.1038/mt.2008.262 
77.	Woodley DT, Cogan J, Mosallaei D, Yim K, Chen M. Characterization of mutant type VII collagens underlying the inversa subtype of recessive dystrophic epidermolysis bullosa. Journal of Dermatological Science. 2021 Nov;104(2):104–11. doi:10.1016/j.jdermsci.2021.09.006 
78.	Chen M, Costa FK, Lindvay CR, Han Y-P, Woodley DT. The recombinant expression of full-length type VII collagen and characterization of molecular mechanisms underlying dystrophic epidermolysis bullosa. Journal of Biological Chemistry. 2002 Jan;277(3):2118–24. doi:10.1074/jbc.m108779200 
79.	Woodley DT, Hou Y, Martin S, Li W, Chen M. Characterization of molecular mechanisms underlying mutations in dystrophic epidermolysis bullosa using site-directed mutagenesis. Journal of Biological Chemistry. 2008 Jun;283(26):17838–45. doi:10.1074/jbc.m709452200 
80.	Chen M, Marinkovich MP, Jones JCR, O’Toole EA, Li Y-Y, Woodley DT. NC1 domain of type VII (anchoring fibril) collagen binds to the Β3 chain of laminin 5 via a unique subdomain within the fibronectin-like repeats. Journal of Dermatological Science. 1998 Mar;16. doi:10.1016/s0923-1811(98)83142-3 
81.	Aumailley M. Laminins and interaction partners in the architecture of the basement membrane at the dermal‐Epidermal Junction. Experimental Dermatology. 2020 Dec 11;30(1):17–24. doi:10.1111/exd.14239 
82.	Calonge MJ, Seoane J, Massagué J. Opposite Smad and chicken ovalbumin upstream promoter transcription factor inputs in the regulation of the collagen VII gene promoter by transforming growth factor-β. Journal of Biological Chemistry. 2004 May;279(22):23759–65. doi:10.1074/jbc.m402178200 
83.	Akasaka E, Kleiser S, Sengle G, Bruckner-Tuderman L, Nyström A. Diversity of mechanisms underlying latent TGF-β activation in recessive dystrophic epidermolysis bullosa. Journal of Investigative Dermatology. 2021 Jun;141(6). doi:10.1016/j.jid.2020.10.024 
84.	Nyström A, Velati D, Mittapalli VR, Fritsch A, Kern JS, Bruckner-Tuderman L. Collagen VII plays a dual role in wound healing. Journal of Clinical Investigation. 2013 Jul 8;123(8):3498–509. doi:10.1172/jci68127 
85.	Fritsch A, Loeckermann S, Kern JS, Braun A, Bösl MR, Bley TA, et al. A hypomorphic mouse model of dystrophic epidermolysis bullosa reveals mechanisms of disease and response to fibroblast therapy. Journal of Clinical Investigation. 2008 May 1;118(5):1669–79. doi:10.1172/jci34292 
86.	Vanden Oever M, Muldoon D, Mathews W, Tolar J. Fludarabine modulates expression of type VII collagen during haematopoietic stem cell transplantation for recessive dystrophic epidermolysis bullosa*. British Journal of Dermatology. 2021 Mar 8;185(2):380–90. doi:10.1111/bjd.19757 
87.	Martins VL, Vyas JJ, Chen M, Purdie K, Mein CA, South AP, et al. Increased invasive behaviour in cutaneous squamous cell carcinoma with loss of basement-membrane type VII collagen. Journal of Cell Science. 2009 Jun 1;122(11):1788–99. doi:10.1242/jcs.042895 
88.	Hashimoto T, Ishii N, Tsuruta D. Production of neoepitopes by dynamic structural changes on BP180/type XVII collagen. Journal of Investigative Dermatology. 2017 Dec;137(12):2462–4. doi:10.1016/j.jid.2017.09.001 
89.	Jacków J, Löffek S, Nyström A, Bruckner-Tuderman L, Franzke C-W. Collagen XVII shedding suppresses re-epithelialization by directing keratinocyte migration and dampening mtor signaling. Journal of Investigative Dermatology. 2016 May;136(5):1031–41. doi:10.1016/j.jid.2016.01.012 
90.	Gatseva A, Sin YY, Brezzo G, Van Agtmael T. Basement membrane collagens and disease mechanisms. Essays in Biochemistry. 2019 Aug 6;63(3):297–312. doi:10.1042/ebc20180071 
91.	Borradori L, Chavanas S, Schaapveld RQJ, Gagnoux-Palacios L, Calafat J, Meneguzzi G, et al. Role of the bullous pemphigoid antigen 180 (BP180) in the Assembly of hemidesmosomes and cell adhesion—reexpression of Bp180 in generalized atrophic benign epidermolysis bullosa keratinocytes. Experimental Cell Research. 1998 Mar;239(2):463–76. doi:10.1006/excr.1997.3923 
92.	Nishimura M, Nishie W, Shirafuji Y, Shinkuma S, Natsuga K, Nakamura H, et al. Extracellular cleavage of collagen XVII is essential for correct cutaneous basement membrane formation. Human Molecular Genetics. 2015 Nov 24;25(2):328–39. doi:10.1093/hmg/ddv478 
93.	Franzke C-W, Tasanen K, Schäcke H, Zhou Z, Tryggvason K, Mauch C, et al. Transmembrane collagen XVII, an epithelial adhesion protein, is shed from the cell surface by Adams. The EMBO Journal. 2002 Oct 1;21(19):5026–35. doi:10.1093/emboj/cdf532 
94.	Franzke C-W, Tasanen K, Schäcke H, Zhou Z, Tryggvason K, Mauch C, et al. Transmembrane collagen XVII, an epithelial adhesion protein, is shed from the cell surface by Adams. The EMBO Journal. 2002 Oct 1;21(19):5026–35. doi:10.1093/emboj/cdf532 
95.	Löffek S, Hurskainen T, Jackow J, Sigloch FC, Schilling O, Tasanen K, et al. Transmembrane collagen XVII modulates integrin dependent keratinocyte migration via PI3K/rac1 signaling. PLoS ONE. 2014 Feb 5;9(2). doi:10.1371/journal.pone.0087263 
96.	Tasanen K, Tunggal L, Chometon G, Bruckner-Tuderman L, Aumailley M. Keratinocytes from patients lacking collagen XVII display a migratory phenotype. The American Journal of Pathology. 2004 Jun;164(6):2027–38. doi:10.1016/s0002-9440(10)63762-5 
97.	Xiang Y, Liu Y, Yang Y, Yan Y, Kim AJ, Guo C, et al. Reduced expression of collagen 17A1 in naturally aged, photoaged, and UV-irradiated human skin in vivo: Potential links to epidermal aging. Journal of Cell Communication and Signaling. 2022 Jan 21;16(3):421–32. doi:10.1007/s12079-021-00654-y 
98.	Watanabe M. Type XVII collagen suppresses interfollicular epidermal proliferation in neonatal and aged skin,  and helps rejuvenate epidermis. 2017 Sept 13; doi:10.26226/morressier.595a9c53d462b80296c9faa2 
99.	Kretzschmar K, Watt FM. Markers of epidermal stem cell subpopulations in adult mammalian skin. Cold Spring Harbor Perspectives in Medicine. 2014 Jul 3;4(10). doi:10.1101/cshperspect.a013631 
100.	Chen J, Zhang M. The PAR3/par6/apkc complex and epithelial cell polarity. Experimental Cell Research. 2013 Jun;319(10):1357–64. doi:10.1016/j.yexcr.2013.03.021 
101.	Watanabe M, Kosumi H, Osada S, Takashima S, Wang Y, Nishie W, et al. Type XVII collagen interacts with the apkc‐par complex and maintains epidermal cell polarity. Experimental Dermatology. 2020 Oct 6;30(1):62–7. doi:10.1111/exd.14196
102.	Liu N, Matsumura H, Kato T, Ichinose S, Takada A, Namiki T, et al. Stem cell competition orchestrates skin homeostasis and ageing. Nature. 2019 Apr 3;568(7752):344–50. doi:10.1038/s41586-019-1085-7 
103.	Liu N, Matsumura H, Kato T, Ichinose S, Takada A, Namiki T, et al. Stem cell competition orchestrates skin homeostasis and ageing. Nature. 2019 Apr 3;568(7752):344–50. doi:10.1038/s41586-019-1085-7 
104.	Matsumura H, Mohri Y, Morinaga H, Fukuda M, Kurata S, Nishimura EK. Hair follicle aging is driven by transepidermal elimination of stem cells via COL17A1 proteolysis. Journal of Dermatological Science. 2017 May;86(2). doi:10.1016/j.jdermsci.2017.02.156 
105.	Jones VA, Patel PM, Gibson FT, Cordova A, Amber KT. The role of collagen XVII in cancer: Squamous cell carcinoma and beyond. Frontiers in Oncology. 2020 Mar 19;10. doi:10.3389/fonc.2020.00352 
106.	Jeong S, Yoon S, Kim S, Jung J, Kor M, Shin K, et al. Anti-wrinkle benefits of peptides complex stimulating skin basement membrane proteins expression. International Journal of Molecular Sciences. 2019 Dec 20;21(1):73. doi:10.3390/ijms21010073 
107.	Aumailley M, Has C, Tunggal L, Bruckner-Tuderman L. Molecular basis of inherited skin-blistering disorders, and therapeutic implications. Expert Reviews in Molecular Medicine. 2006 Oct;8(24):1–21. doi:10.1017/s1462399406000123 
108.	Has C, Bauer JW, Bodemer C, Bolling MC, Bruckner‐Tuderman L, Diem A, et al. Consensus reclassification of inherited Epidermolysis Bullosa and other disorders with skin fragility. British Journal of Dermatology. 2020 Mar 11;183(4):614–27. doi:10.1111/bjd.18921 
109.	Mendiratta V, Malik M. Kindler syndrome. Indian Pediatrics. 2018 Jan;55(1):85–85. doi:10.1007/s13312-018-1239-y 
110.	Ishiko A. Faculty opinions recommendation of inherited epidermolysis bullosa: Updated recommendations on diagnosis and classification. Faculty Opinions – Post-Publication Peer Review of the Biomedical Literature. 2014 Apr 17; doi:10.3410/f.718335617.793493821 
111.	Bauer JW, Lanschuetzer C. Type XVII collagen gene mutations in junctional epidermolysis bullosa and prospects for gene therapy. Clinical and Experimental Dermatology. 2003 Jan;28(1):53–60. doi:10.1046/j.1365-2230.2003.01192.x 
112.	Nakamura H, Sawamura D, Goto M, Nakamura H, Kida M, Ariga T, et al. Analysis of the COL17A1 in non-Herlitz junctional epidermolysis bullosa and amelogenesis imperfecta. International Journal of Molecular Medicine. 2006 Aug 1; doi:10.3892/ijmm.18.2.333 
113.	Giusti D, Gatouillat G, Le Jan S, Plée J, Bernard P, Antonicelli F, et al. Anti-type VII collagen antibodies are identified in a subpopulation of bullous pemphigoid patients with relapse. Frontiers in Immunology. 2018 Mar 21;9. doi:10.3389/fimmu.2018.00570 
114.	Vorobyev A, Ujiie H, Recke A, Buijsrogge JJA, Jonkman MF, Pas HH, et al. Autoantibodies to multiple epitopes on the non-collagenous-1 domain of type VII collagen induce blisters. Journal of Investigative Dermatology. 2015 Jun;135(6):1565–73. doi:10.1038/jid.2015.51 
115.	Matsumura K, Amagai M, Nishikawa T, Hashimoto T. The majority of bullous pemphigoid and herpes gestationis serum samples react with the NC16A domain of the 180-KDA bullous pemphigoid antigen. Archives of Dermatological Research. 1996 Aug;288(9):507–9. doi:10.1007/bf02505245 
116.	Dainichi T, Nishie W, Yamagami Y, Sonobe H, Ujiie H, Kaku Y, et al. Bullous pemphigoid suggestive of complement‐independent Blister Formation with anti‐BP180 IgG4 autoantibodies. British Journal of Dermatology. 2016 Mar 18;175(1):187–90. doi:10.1111/bjd.14411 
117.	Gupta R, Woodley DT, Chen M. Epidermolysis Bullosa Acquisita. Clinics in Dermatology. 2012 Jan;30(1):60–9. doi:10.1016/j.clindermatol.2011.03.011 
118.	Chandrasekaran A, Moser JC. Cutaneous squamous cell carcinoma in epidermolysis bullosa: A review of pathogenesis, diagnosis and management. Cancers. 2025 Oct 1;17(19):3211. doi:10.3390/cancers17193211 
119.	Cho S, Irianto J, Discher DE. Mechanosensing by the nucleus: From pathways to scaling relationships. Journal of Cell Biology. 2017 Jan 2;216(2):305–15. doi:10.1083/jcb.201610042 
120.	Chang Y-C, Wu J-W, Wang C-W, Jang AC-C. Hippo signaling-mediated mechanotransduction in cell movement and cancer metastasis. Frontiers in Molecular Biosciences. 2020 Jan 31;6. doi:10.3389/fmolb.2019.00157 
121.	Miralles F, Posern G, Zaromytidou A-I, Treisman R. Actin dynamics control SRF activity by regulation of its coactivator mal. Cell. 2003 May;113(3):329–42. doi:10.1016/s0092-8674(03)00278-2 
122.	Yang C, Tibbitt MW, Basta L, Anseth KS. Mechanical memory and dosing influence stem cell fate. Nature Materials. 2014 Mar 16;13(6):645–52. doi:10.1038/nmat3889 
123.	Plessner M, Melak M, Chinchilla P, Baarlink C, Grosse R. Nuclear F-actin formation and reorganization upon cell spreading. Journal of Biological Chemistry. 2015 May;290(18):11209–16. doi:10.1074/jbc.m114.627166 
124.	Connelly JT, Gautrot JE, Trappmann B, Tan DW-M, Donati G, Huck WTS, et al. Actin and serum response factor transduce physical cues from the microenvironment to regulate epidermal stem cell fate decisions. Nature Cell Biology. 2010 Jun 27;12(7):711–8. doi:10.1038/ncb2074 
125.	Totaro A, Castellan M, Battilana G, Zanconato F, Azzolin L, Giulitti S, et al. Yap/taz link cell mechanics to notch signalling to control epidermal stem cell fate. Nature Communications. 2017 May 17;8(1). doi:10.1038/ncomms15206 
126.	Totaro A, Castellan M, Battilana G, Zanconato F, Azzolin L, Giulitti S, et al. Yap/taz link cell mechanics to notch signalling to control epidermal stem cell fate. Nature Communications. 2017 May 17;8(1). doi:10.1038/ncomms15206 
127.	Wang N, Tytell JD, Ingber DE. Mechanotransduction at a distance: Mechanically coupling the extracellular matrix with the nucleus. Nature Reviews Molecular Cell Biology. 2009 Jan;10(1):75–82. doi:10.1038/nrm2594 
128.	Miroshnikova YA, Wickström SA. Mechanical forces in nuclear organization. Cold Spring Harbor Perspectives in Biology. 2021 Jun 29;14(1). doi:10.1101/cshperspect.a039685 
129.	Hoffman LM, Smith MA, Jensen CC, Yoshigi M, Blankman E, Ullman KS, et al. Mechanical stress triggers nuclear remodeling and the formation of transmembrane actin nuclear lines with associated nuclear pore complexes. Molecular Biology of the Cell. 2020 Jul 21;31(16):1774–87. doi:10.1091/mbc.e19-01-0027 
130.	Chung H, Oh S, Shin H-W, Lee Y, Lee H, Seok SH. Matrix stiffening enhances DNCB-induced IL-6 secretion in keratinocytes through activation of ERK and PI3K/akt pathway. Frontiers in Immunology. 2021 Nov 11;12. doi:10.3389/fimmu.2021.759992 
131.	De Magistris P, Antonin W. The dynamic nature of the nuclear envelope. Current Biology. 2018 Apr;28(8). doi:10.1016/j.cub.2018.01.073 
132.	Pathak RU, Soujanya M, Mishra RK. Deterioration of nuclear morphology and architecture: A hallmark of senescence and aging. Ageing Research Reviews. 2021 May;67:101264. doi:10.1016/j.arr.2021.101264 
133.	Sahabandu N, Okada K, Khan A, Elnatan D, Starr DA, Ori-McKenney KM, et al. Active microtubule-actin cross-talk mediated by a Nesprin-2G–Kinesin Complex. Science Advances. 2025 Feb 21;11(8). doi:10.1126/sciadv.adq4726 
134.	Sohrabi Molina M, Brauer E, Günther R, Diederich S, Petersen A. Increased substrate stiffness disrupts nuclear-cytoskeletal mechanical coupling in senescent cells. Materials Today Bio. 2025 Dec;35:102472. doi:10.1016/j.mtbio.2025.102472 
135.	Lammerding J, Wolf K. Nuclear envelope rupture: Actin fibers are putting the squeeze on the nucleus. Journal of Cell Biology. 2016 Oct 3;215(1):5–8. doi:10.1083/jcb.201609102 
136.	Khatau SB, Hale CM, Stewart-Hutchinson PJ, Patel MS, Stewart CL, Searson PC, et al. A perinuclear actin cap regulates nuclear shape. Proceedings of the National Academy of Sciences. 2009 Nov 10;106(45):19017–22. doi:10.1073/pnas.0908686106 
137.	Kalukula Y, Stephens AD, Lammerding J, Gabriele S. Mechanics and functional consequences of nuclear deformations. Nature Reviews Molecular Cell Biology. 2022 May 5;23(9):583–602. doi:10.1038/s41580-022-00480-z 
138.	Yonemura S, Wada Y, Watanabe T, Nagafuchi A, Shibata M. Α-catenin as a tension transducer that induces adherens junction development. Nature Cell Biology. 2010 May 9;12(6):533–42. doi:10.1038/ncb2055 
139.	Leckband DE, de Rooij J. Cadherin adhesion and mechanotransduction. Annual Review of Cell and Developmental Biology. 2014 Oct 11;30(1):291–315. doi:10.1146/annurev-cellbio-100913-013212 
140.	Ketema M, Kreft M, Secades P, Janssen H, Sonnenberg A. Nesprin-3 connects plectin and vimentin to the nuclear envelope of Sertoli cells but is not required for Sertoli cell function in spermatogenesis. Molecular Biology of the Cell. 2013 Aug;24(15):2454–66. doi:10.1091/mbc.e13-02-0100 
141.	Almeida FV, Walko G, McMillan JR, McGrath JA, Wiche G, Barber AH, et al. The cytolinker plectin regulates nuclear mechanotransduction in keratinocytes. Journal of Cell Science. 2015 Dec 15;128(24):4475–86. doi:10.1242/jcs.173435 
142.	Maurer M, Lammerding J. The Driving Force: Nuclear Mechanotransduction in cellular function, fate, and disease. Annual Review of Biomedical Engineering. 2019 Jun 4;21(1):443–68. doi:10.1146/annurev-bioeng-060418-052139 
143.	Janin A, Gache V. Nesprins and lamins in health and diseases of cardiac and skeletal muscles. Frontiers in Physiology. 2018 Sept 7;9. doi:10.3389/fphys.2018.01277 
144.	Lüke Y, Zaim H, Karakesisoglou I, Jaeger VM, Sellin L, Lu W, et al. Nesprin-2 Giant (nuance) maintains nuclear envelope architecture and composition in Skin. Journal of Cell Science. 2008 Jun 1;121(11):1887–98. doi:10.1242/jcs.019075 
145.	Haque F, Mazzeo D, Patel JT, Smallwood DT, Ellis JA, Shanahan CM, et al. Mammalian sun protein interaction networks at the inner nuclear membrane and their role in laminopathy disease processes. Journal of Biological Chemistry. 2010 Jan;285(5):3487–98. doi:10.1074/jbc.m109.071910 
146.	Janin A, Gache V. Nesprins and lamins in health and diseases of cardiac and skeletal muscles. Frontiers in Physiology. 2018 Sept 7;9. doi:10.3389/fphys.2018.01277 
147.	Meinke P, Mattioli E, Haque F, Antoku S, Columbaro M, Straatman KR, et al. Muscular dystrophy-associated SUN1 and SUN2 variants disrupt nuclear-cytoskeletal connections and Myonuclear organization. PLoS Genetics. 2014 Sept 11;10(9). doi:10.1371/journal.pgen.1004605 
148.	Carley E, Stewart RK, Zieman A, Jalilian I, King DianeE, Zubek A, et al. The LINC complex transmits integrin-dependent tension to the nuclear lamina and represses epidermal differentiation. 2020 May 4; doi:10.1101/2020.05.03.075085 
149.	Luxton GW, Gomes ER, Folker ES, Vintinner E, Gundersen GG. Linear arrays of nuclear envelope proteins harness retrograde actin flow for nuclear movement. Science. 2010 Aug 20;329(5994):956–9. doi:10.1126/science.1189072 
150.	Lombardi ML, Jaalouk DE, Shanahan CM, Burke B, Roux KJ, Lammerding J. The interaction between Nesprins and Sun proteins at the nuclear envelope is critical for force transmission between the nucleus and cytoskeleton. Journal of Biological Chemistry. 2011 Jul;286(30):26743–53. doi:10.1074/jbc.m111.233700 
151.	Elosegui-Artola A, Andreu I, Beedle AEM, Lezamiz A, Uroz M, Kosmalska AJ, et al. Force triggers yap nuclear entry by regulating transport across nuclear pores. Cell. 2017 Nov;171(6). doi:10.1016/j.cell.2017.10.008 
152.	Stewart RM, Zubek AE, Rosowski KA, Schreiner SM, Horsley V, King MC. Nuclear–cytoskeletal linkages facilitate cross talk between the nucleus and intercellular adhesions. Journal of Cell Biology. 2015 May 11;209(3):403–18. doi:10.1083/jcb.201502024 
153.	Short B. Sun protection for the skin. Journal of Cell Biology. 2015 May 11;209(3):319–319. doi:10.1083/jcb.2093if 
154.	Carley E, Stewart RK, Zieman A, Jalilian I, King DianeE, Zubek A, et al. The LINC complex transmits integrin-dependent tension to the nuclear lamina and represses epidermal differentiation. 2020 May 4; doi:10.1101/2020.05.03.075085 
155.	Mu X, Tseng C, Hambright WS, Matre P, Lin C, Chanda P, et al. Cytoskeleton stiffness regulates cellular senescence and innate immune response in Hutchinson–Gilford Progeria syndrome. Aging Cell. 2020 Jul 25;19(8). doi:10.1111/acel.13152 
156.	Chen C-Y, Chi Y-H, Mutalif RA, Starost MF, Myers TG, Anderson SA, et al. Accumulation of the inner nuclear envelope protein SUN1 is pathogenic in progeric and dystrophic laminopathies. Cell. 2012 Apr;149(3):565–77. doi:10.1016/j.cell.2012.01.059 
157.	Chen C-Y, Chi Y-H, Mutalif RA, Starost MF, Myers TG, Anderson SA, et al. Accumulation of the inner nuclear envelope protein SUN1 is pathogenic in progeric and dystrophic laminopathies. Cell. 2012 Apr;149(3):565–77. doi:10.1016/j.cell.2012.01.059 
158.	Dittmer TA, Misteli T. The lamin protein family. Genome Biology. 2011;12(5):222. doi:10.1186/gb-2011-12-5-222 
159.	Dittmer TA, Misteli T. The lamin protein family. Genome Biology. 2011;12(5):222. doi:10.1186/gb-2011-12-5-222 
160.	Swift J, Discher DE. The nuclear lamina is mechano-responsive to ECM elasticity in mature tissue. Journal of Cell Science. 2014 Jan 1; doi:10.1242/jcs.149203 
161.	Buxboim A, Swift J, Irianto J, Spinler KR, Dingal PCDP, Athirasala A, et al. Matrix elasticity regulates lamin-A,C phosphorylation and turnover with feedback to Actomyosin. Current Biology. 2014 Aug;24(16):1909–17. doi:10.1016/j.cub.2014.07.001 
162.	Martin C, Chen S, Maya-Mendoza A, Lovric J, Sims PF, Jackson DA. Lamin B1 maintains the functional plasticity of Nucleoli. Journal of Cell Science. 2009 May 15;122(10):1551–62. doi:10.1242/jcs.046284 
163.	Lammerding J, Fong LG, Ji JY, Reue K, Stewart CL, Young SG, et al. Lamins A and C but not lamin B1 Regulate Nuclear Mechanics. Journal of Biological Chemistry. 2006 Sept;281(35):25768–80. doi:10.1074/jbc.m513511200 
164.	Vahabikashi A, Sivagurunathan S, Nicdao FA, Han YL, Park CY, Kittisopikul M, et al. Nuclear lamin isoforms differentially contribute to LINC complex-dependent nucleocytoskeletal coupling and whole-cell mechanics. Proceedings of the National Academy of Sciences. 2022 Apr 19;119(17). doi:10.1073/pnas.2121816119 
165.	Rogerson C, Wotherspoon D, O’Shaughnessy RF. Akt1-associated actomyosin remodelling is required for nuclear lamina dispersal and nuclear shrinkage in epidermal terminal differentiation. 2021 Dec 8; doi:10.1101/868034 
166.	Shimi T, Pfleghaar K, Kojima S, Pack C-G, Solovei I, Goldman AE, et al. The A- and B-type nuclear lamin networks: Microdomains involved in chromatin organization and transcription. Genes &amp; Development. 2008 Dec 15;22(24):3409–21. doi:10.1101/gad.1735208 
167.	Finlan LE, Sproul D, Thomson I, Boyle S, Kerr E, Perry P, et al. Recruitment to the nuclear periphery can alter expression of genes in human cells. PLoS Genetics. 2008 Mar 21;4(3). doi:10.1371/journal.pgen.1000039 
168.	Kumaran RI, Spector DL. A genetic locus targeted to the nuclear periphery in living cells maintains its transcriptional competence. The Journal of Cell Biology. 2008 Jan 14;180(1):51–65. doi:10.1083/jcb.200706060 
169.	Reddy KL, Zullo JM, Bertolino E, Singh H. Transcriptional repression mediated by repositioning of genes to the nuclear lamina. Nature. 2008 Feb 13;452(7184):243–7. doi:10.1038/nature06727 
170.	Pascual-Reguant L, Blanco E, Galan S, Le Dily F, Cuartero Y, Serra-Bardenys G, et al. Lamin B1 mapping reveals the existence of dynamic and functional euchromatin lamin B1 domains. Nature Communications. 2018 Aug 24;9(1). doi:10.1038/s41467-018-05912-z 
171.	Kind J, Pagie L, Ortabozkoyun H, Boyle S, de Vries SS, Janssen H, et al. Single-cell dynamics of genome-nuclear lamina interactions. Cell. 2013 Mar;153(1):178–92. doi:10.1016/j.cell.2013.02.028 
172.	Robson MI, Ringel AR, Mundlos S. Regulatory landscaping: How enhancer-promoter communication is sculpted in 3D. Molecular Cell. 2019 Jun;74(6):1110–22. doi:10.1016/j.molcel.2019.05.032 
173.	Reddy KL, Zullo JM, Bertolino E, Singh H. Transcriptional repression mediated by repositioning of genes to the nuclear lamina. Nature. 2008 Feb 13;452(7184):243–7. doi:10.1038/nature06727 
174.	Kumaran RI, Spector DL. A genetic locus targeted to the nuclear periphery in living cells maintains its transcriptional competence. The Journal of Cell Biology. 2008 Jan 14;180(1):51–65. doi:10.1083/jcb.200706060 
175.	van Steensel B, Belmont AS. Lamina-associated domains: Links with chromosome architecture, heterochromatin, and gene repression. Cell. 2017 May;169(5):780–91. doi:10.1016/j.cell.2017.04.022 
176.	Meuleman W, Peric-Hupkes D, Kind J, Beaudry J-B, Pagie L, Kellis M, et al. Constitutive nuclear lamina–genome interactions are highly conserved and associated with a/T-rich sequence. Genome Research. 2012 Nov 2;23(2):270–80. doi:10.1101/gr.141028.112 
177.	Marullo F, Cesarini E, Antonelli L, Gregoretti F, Oliva G, Lanzuolo C. Nucleoplasmic lamin A/C and Polycomb Group of proteins: An evolutionarily conserved interplay. Nucleus. 2016 Mar;7(2):103–11. doi:10.1080/19491034.2016.1157675 
178.	Sen Gupta A, Sengupta K. Lamin B2 modulates nucleolar morphology, dynamics, and function. Molecular and Cellular Biology. 2017 Dec 1;37(24). doi:10.1128/mcb.00274-17 
179.	Kuga T, Nie H, Kazami T, Satoh M, Matsushita K, Nomura F, et al. Lamin B2 prevents chromosome instability by ensuring proper mitotic chromosome segregation. Oncogenesis. 2014 Mar 17;3(3). doi:10.1038/oncsis.2014.6 
180.	Han L, Choudhury S, Mich-Basso JD, Ammanamanchi N, Ganapathy B, Suresh S, et al. Lamin B2 levels regulate polyploidization of cardiomyocyte nuclei and myocardial regeneration. Developmental Cell. 2020 Apr;53(1). doi:10.1016/j.devcel.2020.01.030 
181.	Sen GL, Boxer LD, Webster DE, Bussat RT, Qu K, Zarnegar BJ, et al. ZNF750 is a P63 target gene that induces KLF4 to drive terminal epidermal differentiation. Developmental Cell. 2012 Mar;22(3):669–77. doi:10.1016/j.devcel.2011.12.001 
182.	Li Y, Zhu J, Yu Z, Li H, Jin X. The role of lamin B2 in human diseases. Gene. 2023 Jun;870:147423. doi:10.1016/j.gene.2023.147423 
183.	Dubińska-Magiera M, Kozioł K, Machowska M, Piekarowicz K, Filipczak D, Rzepecki R. Emerin is required for proper nucleus reassembly after mitosis: Implications for new pathogenetic mechanisms for laminopathies detected in EDMD1 patients. Cells. 2019 Mar 13;8(3):240. doi:10.3390/cells8030240 
184.	Fernandez A, Bautista M, Wu L, Pinaud F. Emerin self-assembly and nucleoskeletal coupling regulate nuclear envelope mechanics against stress. 2021 Feb 15; doi:10.1101/2021.02.12.429834 
185.	Tsekrekou M, Stratigi K, Chatzinikolaou G. The nucleolus: In Genome Maintenance and Repair. International Journal of Molecular Sciences. 2017 Jul 1;18(7):1411. doi:10.3390/ijms18071411 
186.	Lafontaine DL, Riback JA, Bascetin R, Brangwynne CP. The nucleolus as a multiphase liquid condensate. Nature Reviews Molecular Cell Biology. 2020 Sept 1;22(3):165–82. doi:10.1038/s41580-020-0272-6 
187.	Ma N, Matsunaga S, Takata H, Ono-Maniwa R, Uchiyama S, Fukui K. Nucleolin functions in nucleolus formation and chromosome congression. Journal of Cell Science. 2007 Jun 15;120(12):2091–105. doi:10.1242/jcs.008771 
188.	Corman A, Sirozh O, Lafarga V, Fernandez-Capetillo O. Targeting the nucleolus as a therapeutic strategy in human disease. Trends in Biochemical Sciences. 2023 Mar;48(3):274–87. doi:10.1016/j.tibs.2022.09.006 
189.	Mao YS, Zhang B, Spector DL. Biogenesis and function of Nuclear Bodies. Trends in Genetics. 2011 Aug;27(8):295–306. doi:10.1016/j.tig.2011.05.006 
190.	Jackson JA, Romeo N, Mietke A, Burns KJ, Totz JF, Martin AC, et al. Scaling behaviour and control of nuclear wrinkling. Nature Physics. 2023 Sept 18;19(12):1927–35. doi:10.1038/s41567-023-02216-y 
191.	Stephens AD, Banigan EJ, Marko JF. Chromatin’s physical properties shape the nucleus and its functions. Current Opinion in Cell Biology. 2019 Jun;58:76–84. doi:10.1016/j.ceb.2019.02.006 
192.	Wang AS, Ong PF, Chojnowski A, Clavel C, Dreesen O. Loss of lamin B1 is a biomarker to quantify cellular senescence in photoaged skin. Scientific Reports. 2017 Nov 15;7(1). doi:10.1038/s41598-017-15901-9 
193.	Wang AS, Ong PF, Chojnowski A, Clavel C, Dreesen O. Loss of lamin B1 is a biomarker to quantify cellular senescence in photoaged skin. Scientific Reports. 2017 Nov 15;7(1). doi:10.1038/s41598-017-15901-9 
194.	Folker ES, Östlund C, Luxton GW, Worman HJ, Gundersen GG. Lamin a variants that cause striated muscle disease are defective in anchoring transmembrane actin-associated nuclear lines for nuclear movement. Proceedings of the National Academy of Sciences. 2010 Dec 20;108(1):131–6. doi:10.1073/pnas.1000824108 
195.	Chen Z-J, Wang W-P, Chen Y-C, Wang J-Y, Lin W-H, Tai L-A, et al. Dysregulated interactions between lamin A and SUN1 induce abnormalities in the nuclear envelope and endoplasmic reticulum in progeric laminopathies. Journal of Cell Science. 2014 Apr 15;127(8):1792–804. doi:10.1242/jcs.139683 
196.	Vahabikashi A, Sivagurunathan S, Nicdao FA, Han YL, Park CY, Wong X, et al. Nuclear lamin isoforms differentially contribute to LINC complex-dependent nucleocytoskeletal coupling and whole cell mechanics. 2021 May 13; doi:10.1101/2021.05.12.443683 
197.	Zhang H, Li J, Yu Y, Ren J, Liu Q, Bao Z, et al. Nuclear lamina erosion-induced resurrection of endogenous retroviruses underlies neuronal aging. Cell Reports. 2023 Nov;42(11):113396. doi:10.1016/j.celrep.2023.113396 
198.	Sadaie M, Salama R, Carroll T, Tomimatsu K, Chandra T, Young ARJ, et al. Redistribution of the lamin B1 genomic binding profile affects rearrangement of heterochromatic domains and SAHF formation during senescence. Genes &amp; Development. 2013 Aug 15;27(16):1800–8. doi:10.1101/gad.217281.113 
199.	Freund A, Laberge R-M, Demaria M, Campisi J. Lamin B1 loss is a senescence-associated biomarker. Molecular Biology of the Cell. 2012 Jun;23(11):2066–75. doi:10.1091/mbc.e11-10-0884 
200.	DeBusk FL. The hutchinson-gilford progeria syndrome. The Journal of Pediatrics. 1972 Apr;80(4):697–724. doi:10.1016/s0022-3476(72)80229-4 
201.	Scaffidi P, Gordon L, Misteli T. The cell nucleus and aging: Tantalizing clues and hopeful promises. PLoS Biology. 2005 Nov 15;3(11). doi:10.1371/journal.pbio.0030395 
202.	Dahl KN, Scaffidi P, Islam MF, Yodh AG, Wilson KL, Misteli T. Distinct structural and mechanical properties of the nuclear lamina in Hutchinson–gilford progeria syndrome. Proceedings of the National Academy of Sciences. 2006 Jul 5;103(27):10271–6. doi:10.1073/pnas.0601058103 
203.	Scaffidi P, Misteli T. Lamin A-dependent nuclear defects in human aging. Science. 2006 May 19;312(5776):1059–63. doi:10.1126/science.1127168 
204.	Uitto J. Searching for clues to premature aging. Trends in Endocrinology &amp; Metabolism. 2002 May;13(4):140–1. doi:10.1016/s1043-2760(02)00595-7 
205.	Ashapkin VV, Kutueva LI, Kurchashova SY, Kireev II. Are there common mechanisms between the hutchinson–gilford progeria syndrome and natural aging? Frontiers in Genetics. 2019 May 15;10. doi:10.3389/fgene.2019.00455 
206.	De Sandre-Giovannoli A, Bernard R, Cau P, Navarro C, Amiel J, Boccaccio I, et al. Lamin a truncation in Hutchinson-Gilford Progeria. Science. 2003 Jun 27;300(5628):2055–2055. doi:10.1126/science.1084125 
207.	Eriksson M, Brown WT, Gordon LB, Glynn MW, Singer J, Scott L, et al. Recurrent de novo point mutations in lamin a cause hutchinson–gilford progeria syndrome. Nature. 2003 Apr 25;423(6937):293–8. doi:10.1038/nature01629 
208.	Skoczyńska A, Budzisz E, Dana A, Rotsztejn H. New look at the role of Progerin in Skin aging. Menopausal Review. 2015;1:53–8. doi:10.5114/pm.2015.49532 
209.	Shumaker DK, Dechat T, Kohlmaier A, Adam SA, Bozovsky MR, Erdos MR, et al. Mutant Nuclear Lamin A leads to progressive alterations of epigenetic control in premature aging. Proceedings of the National Academy of Sciences. 2006 Jun 6;103(23):8703–8. doi:10.1073/pnas.0602569103 
210.	Endisha H, Merrill-Schools J, Zhao M, Bristol M, Wang X, Kubben N, et al. Restoring SIRT6 expression in hutchinson-gilford progeria syndrome cells impedes premature senescence and formation of dysmorphic nuclei. Pathobiology. 2015;82(1):9–20. doi:10.1159/000368856 
211.	Pegoraro G, Kubben N, Wickert U, Göhler H, Hoffmann K, Misteli T. Ageing-related chromatin defects through loss of the Nurd Complex. Nature Cell Biology. 2009 Sept 6;11(10):1261–7. doi:10.1038/ncb1971 
212.	Fernandez P, Scaffidi P, Markert E, Lee J-H, Rane S, Misteli T. Transformation resistance in a premature aging disorder identifies a tumor-protective function of BRD4. Cell Reports. 2014 Oct;9(1):248–60. doi:10.1016/j.celrep.2014.08.069 
213.	Janssen A, Marcelot A, Breusegem S, Legrand P, Zinn-Justin S, Larrieu D. The BAF A12T mutation disrupts lamin A/C interaction, impairing robust repair of nuclear envelope ruptures in Nestor–guillermo progeria syndrome cells. Nucleic Acids Research. 2022 Aug 30;50(16):9260–78. doi:10.1093/nar/gkac726 
214.	Larrieu D, Britton S, Demir M, Rodriguez R, Jackson SP. Chemical inhibition of NAT10 corrects defects of laminopathic cells. Science. 2014 May 2;344(6183):527–32. doi:10.1126/science.1252651 
215.	Balmus G, Larrieu D, Barros AC, Collins C, Abrudan M, Demir M, et al. Targeting of nat10 enhances healthspan in a mouse model of Human Accelerated Aging syndrome. Nature Communications. 2018 Apr 27;9(1). doi:10.1038/s41467-018-03770-3 
216.	Larrieu D, Viré E, Robson S, Breusegem SY, Kouzarides T, Jackson SP. Inhibition of the acetyltransferase NAT10 normalizes progeric and aging cells by rebalancing the Transportin-1 Nuclear Import Pathway. Science Signaling. 2018 Jul 3;11(537). doi:10.1126/scisignal.aar5401 
217.	Hervé S, Scelfo A, Bersano Marchisio G, Grison M, Vaidžiulytė K, Dumont M, et al. Chromosome mis-segregation triggers cell cycle arrest through a mechanosensitive nuclear envelope checkpoint. Nature Cell Biology. 2025 Jan;27(1):73–86. doi:10.1038/s41556-024-01565-x 
218.	Ito S, Horikawa S, Suzuki T, Kawauchi H, Tanaka Y, Suzuki T, et al. Human nat10 is an ATP-dependent RNA acetyltransferase responsible for N4-acetylcytidine formation in 18 S ribosomal RNA (rrna). Journal of Biological Chemistry. 2014 Dec;289(52):35724–30. doi:10.1074/jbc.c114.602698 
219.	Shen Q, Zheng X, McNutt MA, Guang L, Sun Y, Wang J, et al. Nat10, a nucleolar protein, localizes to the midbody and regulates cytokinesis and acetylation of microtubules. Experimental Cell Research. 2009 Jun;315(10):1653–67. doi:10.1016/j.yexcr.2009.03.007 
220.	Mattioli E, Andrenacci D, Garofalo C, Prencipe S, Scotlandi K, Remondini D, et al. Altered modulation of lamin A/C‐HDAC2 interaction and p21 expression during oxidative stress response in HGPS. Aging Cell. 2018 Aug 15;17(5). doi:10.1111/acel.12824 
221.	Liu B, Ghosh S, Yang X, Zheng H, Liu X, Wang Z, et al. Resveratrol rescues Sirt1-dependent adult stem cell decline and alleviates progeroid features in laminopathy-based progeria. Cell Metabolism. 2012 Dec;16(6):738–50. doi:10.1016/j.cmet.2012.11.007 
222.	Ghosh S, Liu B, Zhou Z. Resveratrol activates Sirt1 in a lamin A-dependent manner. Cell Cycle. 2013 Feb 25;12(6):872–6. doi:10.4161/cc.24061 
223.	De Vos WH, Houben F, Kamps M, Malhas A, Verheyen F, Cox J, et al. Repetitive disruptions of the nuclear envelope invoke temporary loss of cellular compartmentalization in Laminopathies. Human Molecular Genetics. 2011 Aug 10;20(21):4175–86. doi:10.1093/hmg/ddr344 
224.	Ho CY, Jaalouk DE, Vartiainen MK, Lammerding J. Lamin A/C and Emerin regulate MKL1–SRF activity by modulating actin dynamics. Nature. 2013 May 5;497(7450):507–11. doi:10.1038/nature12105 
225.	D’Angelo MA, Raices M, Panowski SH, Hetzer MW. Age-dependent deterioration of nuclear pore complexes causes a loss of nuclear integrity in postmitotic cells. Cell. 2009 Jan;136(2):284–95. doi:10.1016/j.cell.2008.11.037 
226.	Waddington CH. The strategy of the genes: a discussion of some aspects of theoretical biology. London: George Allen & Unwin, Ltd.; 1957. 
227.	Goldberg AD, Allis CD, Bernstein E. Epigenetics: A landscape takes shape. Cell. 2007 Feb;128(4):635–8. doi:10.1016/j.cell.2007.02.006 
228.	Henikoff S. The epigenetic landscape: An evolving concept. Frontiers in Epigenetics and Epigenomics. 2023 Mar 20;1. doi:10.3389/freae.2023.1176449 
229.	Takahashi K, Yamanaka S. Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors. Cell. 2006 Aug;126(4):663–76. doi:10.1016/j.cell.2006.07.024 
230.	Ocampo A, Reddy P, Martinez-Redondo P, Platero-Luengo A, Hatanaka F, Hishida T, et al. In vivo amelioration of age-Associated Hallmarks by partial reprogramming. Cell. 2016 Dec;167(7). doi:10.1016/j.cell.2016.11.052 
231.	Sarkar TJ, Quarta M, Mukherjee S, Colville A, Paine P, Doan L, et al. Transient non-integrative nuclear reprogramming promotes multifaceted reversal of aging in human cells. 2019 Mar 18; doi:10.1101/573386 
232.	Grewal SI, Jia S. Heterochromatin revisited. Nature Reviews Genetics. 2007 Jan;8(1):35–46. doi:10.1038/nrg2008 
233.	Misiak B, Ricceri L, Sąsiadek MM. Transposable elements and their epigenetic regulation in mental disorders: Current evidence in the field. Frontiers in Genetics. 2019 Jun 25;10. doi:10.3389/fgene.2019.00580 
234.	McCreery KP, Stubb A, Stephens R, Fursova NA, Cook A, Kruse K, et al. Mechano-osmotic signals control chromatin state and fate transitions in pluripotent stem cells. 2024 Sept 7; doi:10.1101/2024.09.07.611779 
235.	Jung N, Kim T-K. Advances in higher-order chromatin architecture: The move towards 4D genome. BMB Reports. 2021 May 31;54(5):233–45. doi:10.5483/bmbrep.2021.54.5.035 
236.	Jung N, Kim T-K. Advances in higher-order chromatin architecture: The move towards 4D genome. BMB Reports. 2021 May 31;54(5):233–45. doi:10.5483/bmbrep.2021.54.5.035 
237.	Goel VY, Hansen AS. The Macro and micro of chromosome conformation capture. WIREs Developmental Biology. 2020 Sept 28;10(6). doi:10.1002/wdev.395 
238.	Stephens AD, Banigan EJ, Marko JF. Chromatin’s physical properties shape the nucleus and its functions. Current Opinion in Cell Biology. 2019 Jun;58:76–84. doi:10.1016/j.ceb.2019.02.006 
239.	Reyes AA, Marcum RD, He Y. Structure and function of chromatin remodelers. Journal of Molecular Biology. 2021 Jul;433(14):166929. doi:10.1016/j.jmb.2021.166929 
240.	Mardaryev AN, Gdula MR, Yarker JL, Emelianov VN, Poterlowicz K, Sharov AA, et al. P63 and BRG1 control developmentally regulated higher-order chromatin remodelling at the epidermal differentiation complex locus in epidermal progenitor cells. Development. 2014 Jan 1;141(1):101–11. doi:10.1242/dev.103200 
241.	Fessing MY, Mardaryev AN, Gdula MR, Sharov AA, Sharova TY, Rapisarda V, et al. P63 regulates satb1 to control tissue-specific chromatin remodeling during development of the epidermis. Journal of Cell Biology. 2011 Sept 19;194(6):825–39. doi:10.1083/jcb.201101148 
242.	Talbert PB, Henikoff S. Histone variants at a glance. Journal of Cell Science. 2021 Mar 15;134(6). doi:10.1242/jcs.244749 
243.	Hyun K, Jeon J, Park K, Kim J. Writing, erasing and reading histone lysine methylations. Experimental &amp; Molecular Medicine. 2017 Apr 28;49(4). doi:10.1038/emm.2017.11 
244.	Musselman CA, Lalonde M-E, Côté J, Kutateladze TG. Perceiving the epigenetic landscape through histone readers. Nature Structural &amp; Molecular Biology. 2012 Dec;19(12):1218–27. doi:10.1038/nsmb.2436 
245.	You Y, Liang W. SIRT1 and SIRT6: The role in aging-related diseases. Biochimica et Biophysica Acta (BBA) - Molecular Basis of Disease. 2023 Oct;1869(7):166815. doi:10.1016/j.bbadis.2023.166815 
246.	Bielach-Bazyluk A, Zbroch E, Mysliwiec H, Rydzewska-Rosolowska A, Kakareko K, Flisiak I, et al. Sirtuin 1 and skin: Implications in intrinsic and extrinsic aging—a systematic review. Cells. 2021 Apr 6;10(4):813. doi:10.3390/cells10040813 
247.	Satoh A, Brace CS, Rensing N, Cliften P, Wozniak DF, Herzog ED, et al. SIRT1 extends life span and delays aging in mice through the regulation of NK2 Homeobox 1 in the DMH and LH. Cell Metabolism. 2013 Sept;18(3):416–30. doi:10.1016/j.cmet.2013.07.013 
248.	Qiang L, Sample A, Liu H, Wu X, He Y-Y. Epidermal SIRT1 regulates inflammation, cell migration, and wound healing. Scientific Reports. 2017 Oct 26;7(1). doi:10.1038/s41598-017-14371-3 
249.	Blander G, Bhimavarapu A, Mammone T, Maes D, Elliston K, Reich C, et al. SIRT1 promotes differentiation of normal human keratinocytes. Journal of Investigative Dermatology. 2009 Jan;129(1):41–9. doi:10.1038/jid.2008.179 
250.	Ming M, Zhao B, Shea CR, Shah P, Qiang L, White SR, et al. Loss of sirtuin 1 (SIRT1) disrupts skin barrier integrity and sensitizes mice to Epicutaneous Allergen Challenge. Journal of Allergy and Clinical Immunology. 2015 Apr;135(4). doi:10.1016/j.jaci.2014.09.035 
251.	Finkel T, Deng C-X, Mostoslavsky R. Recent progress in the biology and physiology of sirtuins. Nature. 2009 Jul;460(7255):587–91. doi:10.1038/nature08197 
252.	Dicker AJ, Popa C, Dahler AL, Serewko MM, Hilditch-Maguire PA, Frazer IH, et al. E2F-1 induces proliferation-specific genes and suppresses squamous differentiation-specific genes in human epidermal keratinocytes. Oncogene. 2000 Jun 6;19(25):2887–94. doi:10.1038/sj.onc.1203610 
253.	Morris BJ, Willcox DC, Donlon TA, Willcox BJ. foxo3: A major gene for human longevity - a mini-review. Gerontology. 2015;61(6):515–25. doi:10.1159/000375235 
254.	Moon KM, Lee M-K, Hwang T, Choi CW, Kim MS, Kim H-R, et al. The multi-functional roles of forkhead box protein O in skin aging and diseases. Redox Biology. 2021 Oct;46:102101. doi:10.1016/j.redox.2021.102101 
255.	Kawahara TLA, Michishita E, Adler AS, Damian M, Berber E, Lin M, et al. SIRT6 links histone H3 lysine 9 deacetylation to NF-ΚB-dependent gene expression and organismal life span. Cell. 2009 Jan;136(1):62–74. doi:10.1016/j.cell.2008.10.052 
256.	Sharma A, Diecke S, Zhang WY, Lan F, He C, Mordwinkin NM, et al. The role of SIRT6 protein in aging and reprogramming of human induced pluripotent stem cells. Journal of Biological Chemistry. 2013 Jun;288(25):18439–47. doi:10.1074/jbc.m112.405928 
257.	Cheng C, Wang Y, Huo J, Zhang Y, Li R. SIRT6 knockdown alleviates keratinocyte hyperproliferation and inflammation in psoriasis via modulating acetylation of FOXO1. International Immunopharmacology. 2025 Jan;146:113932. doi:10.1016/j.intimp.2024.113932 
258.	Abbotto E, Miro C, Piacente F, Salis A, Murolo M, Nappi A, et al. SIRT6 pharmacological inhibition delays skin cancer progression in the squamous cell carcinoma. Biomedicine &amp; Pharmacotherapy. 2023 Oct;166:115326. doi:10.1016/j.biopha.2023.115326 
259.	Ming M, Han W, Zhao B, Sundaresan NR, Deng C-X, Gupta MP, et al. Data from SIRT6 promotes COX-2 expression and acts as an oncogene in skin cancer. 2023 Mar 30; doi:10.1158/0008-5472.c.6506559
260.	Garcia‐Peterson LM, Guzmán‐Pérez G, Krier CR, Ahmad N. The sirtuin 6: An overture in skin cancer. Experimental Dermatology. 2019 Dec 29;29(2):124–35. doi:10.1111/exd.14057 
261.	Yang JH, Hayano M, Griffin PT, Amorim JA, Bonkowski MS, Apostolides JK, et al. Loss of epigenetic information as a cause of mammalian aging. Cell. 2023 Jan;186(2).  doi: 10.1016/j.cell.2024.01.049
262.	Tian X, Firsanov D, Zhang Z, Cheng Y, Luo L, Tombline G, et al. SIRT6 is responsible for more efficient DNA double-strand break repair in long-lived species. Cell. 2019 Apr;177(3). doi:10.1016/j.cell.2019.03.043 
263.	Simon M, Van Meter M, Ablaeva J, Ke Z, Gonzalez RS, Taguchi T, et al. Line1 derepression in aged wild-type and SIRT6-deficient mice drives inflammation. Cell Metabolism. 2019 Apr;29(4). doi:10.1016/j.cmet.2019.02.014 
264.	Korotkov A, Seluanov A, Gorbunova V. Sirtuin 6: Linking longevity with genome and epigenome stability. Trends in Cell Biology. 2021 Dec;31(12):994–1006. doi:10.1016/j.tcb.2021.06.009 
265.	Ghosh S, Liu B, Wang Y, Hao Q, Zhou Z. Lamin A is an endogenous SIRT6 activator and promotes SIRT6-mediated DNA repair. Cell Reports. 2015 Nov;13(7):1396–406. doi:10.1016/j.celrep.2015.10.006 
266.	Korotkov A, Seluanov A, Gorbunova V. Sirtuin 6: Linking longevity with genome and epigenome stability. Trends in Cell Biology. 2021 Dec;31(12):994–1006. doi:10.1016/j.tcb.2021.06.009 
267.	LeBoeuf M, Terrell A, Trivedi S, Sinha S, Epstein JA, Olson EN, et al. HDAC1 and HDAC2 Act redundantly to control P63 and p53 functions in epidermal progenitor cells. Developmental Cell. 2010 Dec;19(6):807–18. doi:10.1016/j.devcel.2010.10.015 
268.	Hughes MW, Jiang T-X, Lin S-J, Leung Y, Kobielak K, Widelitz RB, et al. Disrupted ectodermal organ morphogenesis in mice with a conditional histone deacetylase 1, 2 deletion in the epidermis. Journal of Investigative Dermatology. 2014 Jan;134(1):24–32. doi:10.1038/jid.2013.283 
269.	Lee Y, Shin MH, Kim M-K, Kim YK, Shin HS, Lee DH, et al. Increased histone acetylation and decreased expression of specific histone deacetylases in ultraviolet-irradiated and intrinsically aged human skin in vivo. International Journal of Molecular Sciences. 2021 Feb 18;22(4):2032. doi:10.3390/ijms22042032 
270.	Szigety KM, Liu F, Yuan CY, Moran DJ, Horrell J, Gochnauer HR, et al. HDAC3 ensures stepwise epidermal stratification via NCOR/SMRT-reliant mechanisms independent of its histone deacetylase activity. Genes &amp; Development. 2020 May 28;34(13–14):973–88. doi:10.1101/gad.333674.119 
271.	Damodaran K, Venkatachalapathy S, Alisafaei F, Radhakrishnan AV, Sharma Jokhun D, Shenoy VB, et al. Compressive force induces reversible chromatin condensation and cell geometry–dependent transcriptional response. Molecular Biology of the Cell. 2018 Dec;29(25):3039–51. doi:10.1091/mbc.e18-04-0256 
272.	Zhang T, Yang L, Ke Y, Lei J, Shen S, Shao S, et al. EZH2-dependent epigenetic modulation of histone H3 lysine-27 contributes to psoriasis by promoting keratinocyte proliferation. Cell Death &amp; Disease. 2020 Oct 3;11(10). doi:10.1038/s41419-020-03028-1 
273.	Wurm S, Zhang J, Guinea-Viniegra J, García F, Muñoz J, Bakiri L, et al. Terminal epidermal differentiation is regulated by the interaction of FRA-2/AP-1 with EZH2 and ERK1/2. Genes &amp; Development. 2014 Dec 29;29(2):144–56. doi:10.1101/gad.249748.114
274.	Ezhkova E, Pasolli HA, Parker JS, Stokes N, Su I, Hannon G, et al. EZH2 orchestrates gene expression for the stepwise differentiation of tissue-specific stem cells. Cell. 2009 Mar;136(6):1122–35. doi:10.1016/j.cell.2008.12.043 
275.	Driskell I, Oda H, Blanco S, Nascimento E, Humphreys P, Frye M. The histone methyltransferase SETD8 acts in concert with c-myc and is required to maintain skin. The EMBO Journal. 2011 Nov 25;31(3):616–29. doi:10.1038/emboj.2011.421 
276.	Li Z, Nie F, Wang S, Li L. Histone H4 lys 20 monomethylation by histone methylase SET8 mediates WNT target gene activation. Proceedings of the National Academy of Sciences. 2011 Jan 31;108(8):3116–23. doi:10.1073/pnas.1009353108 
277.	Sen P, Lan Y, Li CY, Sidoli S, Donahue G, Dou Z, et al. Histone acetyltransferase p300 induces de novo super-enhancers to drive cellular senescence. Molecular Cell. 2019 Feb;73(4). doi:10.1016/j.molcel.2019.01.021 
278.	Di Fede E, Taci E, Castiglioni S, Rebellato S, Ancona S, Grazioli P, et al. P300 inhibition delays premature cellular senescence. npj Aging. 2025 Jul 10;11(1). doi:10.1038/s41514-025-00251-y 
279.	Luis NM, Morey L, Mejetta S, Pascual G, Janich P, Kuebler B, et al. Regulation of human epidermal stem cell proliferation and senescence requires polycomb- dependent and -independent functions of CBX4. Cell Stem Cell. 2011 Nov;9(5):486. doi:10.1016/j.stem.2011.09.014 
280.	Sun J, Gui Y, Zhou S, Zheng X-L. Unlocking the secrets of aging: Epigenetic reader brd4 as the target to combatting aging-related diseases. Journal of Advanced Research. 2024 Sept;63:207–18. doi:10.1016/j.jare.2023.11.006 
281.	Tibben BM, Rothbart SB. Mechanisms of DNA methylation regulatory function and crosstalk with histone lysine methylation. Journal of Molecular Biology. 2024 Apr;436(7):168394. doi:10.1016/j.jmb.2023.168394 
282.	Moore LD, Le T, Fan G. DNA methylation and its basic function. Neuropsychopharmacology. 2012 Jul 11;38(1):23–38. doi:10.1038/npp.2012.112 
283.	Williams K, Christensen J, Helin K. DNA methylation: Tet proteins—guardians of CPG islands? EMBO reports. 2011 Dec 9;13(1):28–35. doi:10.1038/embor.2011.233 
284.	Bienkowska A, Raddatz G, Söhle J, Kristof B, Völzke H, Gallinat S, et al. Development of an epigenetic clock to predict visual age progression of human skin. Frontiers in Aging. 2024 Jan 11;4. doi:10.3389/fragi.2023.1258183 
285.	Bell CG, Lowe R, Adams PD, Baccarelli AA, Beck S, Bell JT, et al. DNA methylation aging clocks: Challenges and recommendations. Genome Biology. 2019 Nov 25;20(1). doi:10.1186/s13059-019-1824-y 
286.	Boroni M, Zonari A, Reis de Oliveira C, Alkatib K, Ochoa Cruz EA, Brace LE, et al. Highly accurate skin-specific methylome analysis algorithm as a platform to screen and validate therapeutics for Healthy Aging. Clinical Epigenetics. 2020 Jul 13;12(1). doi:10.1186/s13148-020-00899-1 
287.	Grönniger E, Weber B, Heil O, Peters N, Stäb F, Wenck H, et al. Aging and chronic sun exposure cause distinct epigenetic changes in human skin. PLoS Genetics. 2010 May 27;6(5). doi:10.1371/journal.pgen.1000971 
288.	Raddatz G, Hagemann S, Aran D, Söhle J, Kulkarni PP, Kaderali L, et al. Aging is associated with highly defined epigenetic changes in the human epidermis. Epigenetics &amp; Chromatin. 2013 Oct 31;6(1). doi:10.1186/1756-8935-6-36 
289.	Banila C, Green D, Katsanos D, Viana J, Osmaston A, Menendez Vazquez A, et al. A noninvasive method for whole-genome skin methylome profiling. British Journal of Dermatology. 2023 Sept 2;189(6):750–9. doi:10.1093/bjd/ljad316 
290.	Jung S-E, Lim SM, Hong SR, Lee EH, Shin K-J, Lee HY. DNA methylation of the ELOVL2, FHL2, KLF14, C1orf132/MIR29B2C, and TRIM59 genes for age prediction from blood, saliva, and buccal swab samples. Forensic Science International: Genetics. 2019 Jan;38:1–8. doi:10.1016/j.fsigen.2018.09.010 
291.	Slieker RC, Relton CL, Gaunt TR, Slagboom PE, Heijmans BT. Age-related DNA methylation changes are tissue-specific with ELOVL2 promoter methylation as exception. Epigenetics &amp; Chromatin. 2018 May 30;11(1). doi:10.1186/s13072-018-0191-3 
292.	Lowe R, Barton C, Jenkins CA, Ernst C, Forman O, Fernandez-Twinn DS, et al. Ageing-associated DNA methylation dynamics are a molecular readout of lifespan variation among mammalian species. Genome Biology. 2018 Feb 16;19(1). doi:10.1186/s13059-018-1397-1 
293.	Rakyan VK, Down TA, Maslau S, Andrew T, Yang T-P, Beyan H, et al. Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains. Genome Research. 2010 Mar 10;20(4):434–9. doi:10.1101/gr.103101.109 
294.	Berman BP, Weisenberger DJ, Aman JF, Hinoue T, Ramjan Z, Liu Y, et al. Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina–associated domains. Nature Genetics. 2011 Nov 27;44(1):40–6. doi:10.1038/ng.969 
295.	Hon GC, Hawkins RD, Caballero OL, Lo C, Lister R, Pelizzola M, et al. Global DNA hypomethylation coupled to repressive chromatin domain formation and gene silencing in breast cancer. Genome Research. 2011 Dec 7;22(2):246–58. doi:10.1101/gr.125872.111 
296.	Guo H, Vuille JA, Wittner BS, Lachtara EM, Hou Y, Lin M, et al. DNA hypomethylation silences anti-tumor immune genes in early prostate cancer and ctcs. Cell. 2023 Jun;186(13). doi:10.1016/j.cell.2023.05.028 
297.	Grönniger E, Weber B, Heil O, Peters N, Stäb F, Wenck H, et al. Aging and chronic sun exposure cause distinct epigenetic changes in human skin. PLoS Genetics. 2010 May 27;6(5). doi:10.1371/journal.pgen.1000971 
298.	Raddatz G, Hagemann S, Aran D, Söhle J, Kulkarni PP, Kaderali L, et al. Aging is associated with highly defined epigenetic changes in the human epidermis. Epigenetics &amp; Chromatin. 2013 Oct 31;6(1). doi:10.1186/1756-8935-6-36 
299.	Bormann F, Rodríguez‐Paredes M, Hagemann S, Manchanda H, Kristof B, Gutekunst J, et al. Reduced                    dna                    methylation patterning and transcriptional connectivity define human skin aging. Aging Cell. 2016 Mar 23;15(3):563–71. doi:10.1111/acel.12470 
300.	Shah PP, Donahue G, Otte GL, Capell BC, Nelson DM, Cao K, et al. Lamin B1 depletion in senescent cells triggers large-scale changes in gene expression and the chromatin landscape. Genes &amp; Development. 2013 Aug 9;27(16):1787–99. doi:10.1101/gad.223834.113 
301.	Sen GL, Reuter JA, Webster DE, Zhu L, Khavari PA. DNMT1 maintains progenitor function in self-renewing somatic tissue. Nature. 2010 Jan 17;463(7280):563–7. doi:10.1038/nature08683 
302.	Köhler F, Rodríguez-Paredes M. DNA methylation in epidermal differentiation, aging, and cancer. Journal of Investigative Dermatology. 2020 Jan;140(1):38–47. doi:10.1016/j.jid.2019.05.011 
303.	Leśniak W. Epigenetic regulation of S100 protein expression. Clinical Epigenetics. 2011 Feb 22;2(2):77–83. doi:10.1007/s13148-011-0023-9 
304.	Andersen B, Weinberg WC, Rennekampff O, McEvilly RJ, Bermingham JR, Hooshmand F, et al. Functions of the POU domain genes SKN-1A/I and TST-1/oct-6/SCIP in epidermal differentiation. Genes &amp; Development. 1997 Jul 15;11(14):1873–84. doi:10.1101/gad.11.14.1873 
305.	Nakamura Y, Kawachi Y, Xu X, Sakurai H, Ishii Y, Takahashi T, et al. The combination of ubiquitous transcription factors AP‐1 and SP1 directs keratinocyte‐specific and differentiation‐specific gene expression in vitro. Experimental Dermatology. 2007 Jan 3;16(2):143–50. doi:10.1111/j.1600-0625.2006.00528.
306.	Eckert RL, Crish JF, Banks EB, Welter JF. The epidermis: Genes on – genes off. Journal of Investigative Dermatology. 1997 Oct;109(4):501–9. doi:10.1111/1523-1747.ep12336477 
307.	Vandiver AR, Irizarry RA, Hansen KD, Garza LA, Runarsson A, Li X, et al. Age and sun exposure-related widespread genomic blocks of hypomethylation in nonmalignant skin. Genome Biology. 2015 Apr 16;16(1). doi:10.1186/s13059-015-0644-y 
308.	Bormann F, Rodríguez‐Paredes M, Hagemann S, Manchanda H, Kristof B, Gutekunst J, et al. Reduced                    dna                    methylation patterning and transcriptional connectivity define human skin aging. Aging Cell. 2016 Mar 23;15(3):563–71. doi:10.1111/acel.12470 
309.	Roos L, Sandling JK, Bell CG, Glass D, Mangino M, Spector TD, et al. Higher nevus count exhibits a distinct DNA methylation signature in healthy human skin: Implications for melanoma. Journal of Investigative Dermatology. 2017 Apr;137(4):910–20. doi:10.1016/j.jid.2016.11.029 
310.	Lu AT, Fei Z, Haghani A, et al. Universal DNA methylation age across mammalian tissues. Nature Aging. 2023;3(9):1166, doi:10.1038/s43587-023-00499-7
311.	Salameti V, Tomas IM, Gunn DA. Methylation levels in keratinocyte basal cells reflect donor age and associate with altered cellular proliferation pathways. Epigenomics. 2025 Jul 6;17(11):711–9. doi:10.1080/17501911.2025.2526320
312.	Rakyan VK, Down TA, Balding DJ, Beck S. EPIGENOME-wide association studies for common human diseases. Nature Reviews Genetics. 2011 Jul 12;12(8):529–41. doi:10.1038/nrg3000
313.	Guelen L, Pagie L, Brasset E, Meuleman W, Faza MB, Talhout W, et al. Domain organization of human chromosomes revealed by mapping of nuclear lamina interactions. Nature. 2008 May 7;453(7197):948–51. doi:10.1038/nature06947 
314.	Kind J, Pagie L, de Vries SS, Nahidiazar L, Dey SS, Bienko M, et al. Genome-wide maps of nuclear lamina interactions in single human cells. Cell. 2015 Sept;163(1):134–47. doi:10.1016/j.cell.2015.08.040 
315.	Sun C, Zhao Y, Guo L, Qiu J, Peng Q. The interplay between histone modifications and nuclear lamina in genome regulation. Journal of Genetics and Genomics. 2025 Jan;52(1):24–38. doi:10.1016/j.jgg.2024.10.005 
316.	Guelen L, Pagie L, Brasset E, Meuleman W, Faza MB, Talhout W, et al. Domain organization of human chromosomes revealed by mapping of nuclear lamina interactions. Nature. 2008 May 7;453(7197):948–51. doi:10.1038/nature06947 
317.	Pujadas EM, Wei X, Acosta N, Carter L, Yang J, Almassalha L, et al. Depletion of lamins B1 and B2 alters chromatin mobility and induces differential gene expression by a mesoscale-motion dependent mechanism. 2023 Jun 26; doi:10.1101/2023.06.26.546573 
318.	Gil L, Niño SA, Chi-Ahumada E, Rodríguez-Leyva I, Guerrero C, Rebolledo AB, et al. Perinuclear lamin A and nucleoplasmic lamin B2 characterize two types of hippocampal neurons through alzheimer’s disease progression. International Journal of Molecular Sciences. 2020 Mar 7;21(5):1841. doi:10.3390/ijms21051841 
319.	Meuleman W, Peric-Hupkes D, Kind J, Beaudry J-B, Pagie L, Kellis M, et al. Constitutive nuclear lamina–genome interactions are highly conserved and associated with a/T-rich sequence. Genome Research. 2012 Nov 2;23(2):270–80. doi:10.1101/gr.141028.112 
320.	Lenain C, de Graaf CA, Pagie L, Visser NL, de Haas M, de Vries SS, et al. Massive reshaping of genome–nuclear lamina interactions during oncogene-induced senescence. Genome Research. 2017 Sept 15;27(10):1634–44. doi:10.1101/gr.225763.117 
321.	Shaban HA, Gasser SM. Dynamic 3D genome reorganization during senescence: Defining Cell states through chromatin. Cell Death &amp; Differentiation. 2023 Aug 18;32(1):9–15. doi:10.1038/s41418-023-01197-y 
322.	Bellanger A, Madsen-Østerbye J, Galigniana NM, Collas P. Restructuring of lamina-associated domains in senescence and cancer. Cells. 2022 Jun 5;11(11):1846. doi:10.3390/cells11111846 
323.	Shah PP, Donahue G, Otte GL, Capell BC, Nelson DM, Cao K, et al. Lamin B1 depletion in senescent cells triggers large-scale changes in gene expression and the chromatin landscape. Genes &amp; Development. 2013 Aug 9;27(16):1787–99. doi:10.1101/gad.223834.113 
324.	Shimi T, Kittisopikul M, Tran J, Goldman AE, Adam SA, Zheng Y, et al. Structural Organization of Nuclear Lamins A, C, B1, and B2 revealed by Superresolution Microscopy. Molecular Biology of the Cell. 2015 Nov 5;26(22):4075–86. doi:10.1091/mbc.e15-07-0461 
325.	Kumari A, Vertii A. Perspective: “Current understanding of nads dynamics and mechanisms of disease.” Gene. 2024 Feb;894:147960. doi:10.1016/j.gene.2023.147960 
326.	Kind J, Pagie L, de Vries SS, Nahidiazar L, Dey SS, Bienko M, et al. Genome-wide maps of nuclear lamina interactions in single human cells. Cell. 2015 Sept;163(1):134–47. doi:10.1016/j.cell.2015.08.040 
327.	Dillinger S, Straub T, Németh A. Nucleolus Association of chromosomal domains is largely maintained in cellular senescence despite massive nuclear reorganisation. PLOS ONE. 2017 Jun 2;12(6). doi:10.1371/journal.pone.0178821 
328.	Chandra T, Ewels PA, Schoenfelder S, Furlan-Magaril M, Wingett SW, Kirschner K, et al. Global reorganization of the nuclear landscape in senescent cells. Cell Reports. 2015 Feb;10(4):471–83. doi:10.1016/j.celrep.2014.12.055 
329.	Buchwalter A, Hetzer MW. Nucleolar expansion and elevated protein translation in premature aging. Nature Communications. 2017 Aug 30;8(1). doi:10.1038/s41467-017-00322-z 
330.	Phan T, Khalid F, Iben S. Nucleolar and ribosomal dysfunction—a common pathomechanism in childhood progerias? Cells. 2019 Jun 4;8(6):534. doi:10.3390/cells8060534 
331.	Paço A, Freitas R, Vieira-da-Silva A. Conversion of DNA sequences: From a transposable element to a tandem repeat or to a gene. Genes. 2019 Dec 5;10(12):1014. doi:10.3390/genes10121014
332.	McClintock B. The origin and behavior of mutable loci in maize. Proceedings of the National Academy of Sciences. 1950 Jun;36(6):344–55. doi:10.1073/pnas.36.6.344 
333.	Prokopov D, Tunbak H, Leddy E, Drylie B, Camera F, Deniz Ö. Transposable elements as genome regulators in normal and malignant haematopoiesis. Blood Cancer Journal. 2025 May 6;15(1). doi:10.1038/s41408-025-01295-9 
334.	Wicker T, Sabot F, Hua-Van A, Bennetzen JL, Capy P, Chalhoub B, et al. A unified classification system for eukaryotic transposable elements. Nature Reviews Genetics. 2007 Dec;8(12):973–82. doi:10.1038/nrg2165 
335.	Mielich K, Shtifman-Segal E, Golz JC, Zeng G, Wang Y, Berman J, et al. Maize transposable elements ac/ds as insertion mutagenesis tools in candida albicans. G3 Genes|Genomes|Genetics. 2018 Apr 1;8(4):1139–45. doi:10.1534/g3.117.300388 
336.	Kassiotis G. Endogenous retroviruses and the development of cancer. The Journal of Immunology. 2014 Feb;192(4):1343–9. doi:10.4049/jimmunol.1302972 
337.	Beck CR, Collier P, Macfarlane C, Malig M, Kidd JM, Eichler EE, et al. Line-1 retrotransposition activity in human genomes. Cell. 2010 Jun;141(7):1159–70. doi:10.1016/j.cell.2010.05.021 
338.	Kaessmann H, Vinckenbosch N, Long M. RNA-based gene duplication: Mechanistic and evolutionary insights. Nature Reviews Genetics. 2009 Jan;10(1):19–31. doi:10.1038/nrg2487 
339.	Kramerov DA, Vassetzky NS. Origin and evolution of Sines in eukaryotic genomes. Heredity. 2011 Jun 15;107(6):487–95. doi:10.1038/hdy.2011.43 
340.	Rowold DJ, Herrera RJ. Alu elements and the human genome. Genetica. 2000 Jan;108(1):57–72. doi:10.1023/a:1004099605261 
341.	Gianfrancesco O, Geary B, Savage AL, Billingsley KJ, Bubb VJ, Quinn JP. The role of Sine-VNTR-Alu (SVA) retrotransposons in shaping the human genome. International Journal of Molecular Sciences. 2019 Nov 27;20(23):5977. doi:10.3390/ijms20235977 
342.	Feschotte C, Pritham EJ. DNA transposons and the evolution of eukaryotic genomes. Annual Review of Genetics. 2007 Dec 1;41(1):331–68. doi:10.1146/annurev.genet.40.110405.090448 
343.	Della Valle F, Reddy P, Yamamoto M, Liu P, Saera-Vila A, Bensaddek D, et al. line-1            RNA causes heterochromatin erosion and is a target for amelioration of senescent phenotypes in progeroid syndromes. Science Translational Medicine. 2022 Aug 10;14(657). doi:10.1126/scitranslmed.abl6057 
344.	Touma F, Lambert M, Martínez Villarreal A, Gantchev J, Ramchatesingh B, Litvinov IV. The ultraviolet irradiation of keratinocytes induces ectopic expression of line-1 retrotransposon machinery and leads to cellular senescence. Biomedicines. 2023 Nov 10;11(11):3017. doi:10.3390/biomedicines11113017 
345.	Min X, Zheng M, Yu Y, Wu J, Kuang Q, Hu Z, et al. Ultraviolet light induces HERV expression to activate rig‐i signalling pathway in keratinocytes. Experimental Dermatology. 2022 Mar 28; doi:10.1111/exd.14568 
346.	Mao J, Zhang Q, Zhuang Y, Zhang Y, Li L, Pan J, et al. Reactivation of senescence-associated endogenous retroviruses by ATF3 drives interferon signaling in aging. Nature Aging. 2024 Nov 14;4(12):1794–812. doi:10.1038/s43587-024-00745-6 
347.	De Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW, et al. L1 drives IFN in senescent cells and promotes age-associated inflammation. Nature. 2019 Feb 6;566(7742):73–8. doi:10.1038/s41586-018-0784-9 
348.	Chen Y, Chen Z, Wang H, Cui Z, Li K-L, Song Z, et al. A CGAS-mediated mechanism in naked mole-rats potentiates DNA repair and delays aging. Science. 2025 Oct 9;390(6769). doi:10.1126/science.adp5056 
349.	Simon M, Van Meter M, Ablaeva J, Ke Z, Gonzalez RS, Taguchi T, et al. Line1 derepression in aged wild-type and SIRT6-deficient mice drives inflammation. Cell Metabolism. 2019 Apr;29(4). doi:10.1016/j.cmet.2019.02.014 
350.	Van Meter M, Kashyap M, Rezazadeh S, Geneva AJ, Morello TD, Seluanov A, et al. SIRT6 represses Line1 retrotransposons by ribosylating KAP1 but this repression fails with stress and age. Nature Communications. 2014 Sept 23;5(1). doi:10.1038/ncomms6011 
351.	Bi S, Liu Z, Wu Z, Wang Z, Liu X, Wang S, et al. SIRT7 antagonizes human stem cell aging as a heterochromatin stabilizer. Protein &amp; Cell. 2020 Jun 6;11(7):483–504. doi:10.1007/s13238-020-00728-4 
352.	Wu Z, Qu J, Zhang W, Liu G-H. Stress, epigenetics, and aging: Unraveling the intricate crosstalk. Molecular Cell. 2024 Jan;84(1):34–54. doi:10.1016/j.molcel.2023.10.006 
353.	Zhao Y, Simon M, Seluanov A, Gorbunova V. DNA damage and repair in age-related inflammation. Nature Reviews Immunology. 2022 Jul 13;23(2):75–89. doi:10.1038/s41577-022-00751-y 
354.	Li X, Yu H, Li D, Liu N. Line-1 transposable element Renaissance in aging and age-related diseases. Ageing Research Reviews. 2024 Sept;100:102440. doi:10.1016/j.arr.2024.102440
355.	Sun T, Xu Y, Angel N, Chen L, Zhang K, Hoffman BD, et al. A subset of transposable elements as Mechano-response enhancer elements in controlling human embryonic stem cell fate. Nature Cell Biology. 2025 Sept 17;27(10):1785–96. doi:10.1038/s41556-025-01770-2 
356.	Cosgrove BD, Bounds LR, Taylor CK, Su AL, Rizzo AJ, Barrera A, et al. Mechanosensitive genomic enhancers potentiate the cellular response to matrix stiffness. 2024 Jan 10; doi:10.1101/2024.01.10.574997 
357.	Booth LN, Brunet A. The aging epigenome. Molecular Cell. 2016 Jun;62(5):728–44. doi:10.1016/j.molcel.2016.05.013 
358.	Kline-Smith SL, Walczak CE. Mitotic spindle assembly and chromosome segregation. Molecular Cell. 2004 Aug;15(3):317–27. doi:10.1016/j.molcel.2004.07.012 
359.	Crosby, Meredith E. Cell cycle: Principles of control. Yale Journal of Biology and Medicine; 2008. 
360.	Beaudouin J, Gerlich D, Daigle N, Eils R, Ellenberg J. Nuclear envelope breakdown proceeds by microtubule-induced tearing of the lamina. Cell. 2002 Jan;108(1):83–96. doi:10.1016/s0092-8674(01)00627-4 
361.	Volkart PA, Bitencourt-Ferreira G, Souto AA, de Azevedo WF. Cyclin-dependent kinase 2 in cellular senescence and cancer. A structural and Functional Review. Current Drug Targets. 2019 May 9;20(7):716–26. doi:10.2174/1389450120666181204165344 
362.	Kumari R, Jat P. Mechanisms of cellular senescence: Cell cycle arrest and senescence associated secretory phenotype. Frontiers in Cell and Developmental Biology. 2021 Mar 29;9. doi:10.3389/fcell.2021.645593 
363.	Belaadi N, Pernet L, Aureille J, Chadeuf G, Rio M, Vaillant N, et al. SUN2 regulates mitotic duration in response to extracellular matrix rigidity. Proceedings of the National Academy of Sciences. 2022 Nov 2;119(45). doi:10.1073/pnas.2116167119 
364.	Xiong H, Rivero F, Euteneuer U, Mondal S, Mana‐Capelli S, Larochelle D, et al. dictyostelium sun‐1 connects the centrosome to chromatin and ensures genome stability. Traffic. 2008 Feb 11;9(5):708–24. doi:10.1111/j.1600-0854.2008.00721
365.	Kind J, Pagie L, Ortabozkoyun H, Boyle S, de Vries SS, Janssen H, et al. Single-cell dynamics of genome-nuclear lamina interactions. Cell. 2013 Mar;153(1):178–92. doi:10.1016/j.cell.2013.02.028 
366.	Bickmore WA, van Steensel B. Genome architecture: Domain organization of interphase chromosomes. Cell. 2013 Mar;152(6):1270–84. doi:10.1016/j.cell.2013.02.001 
367.	van Schaik T, Vos M, Peric-Hupkes D, van Steensel B. Cell cycle dynamics of lamina associated DNA. 2019 Dec 19; doi:10.1101/2019.12.19.881979 
368.	Hiratani I, Ryba T, Itoh M, Yokochi T, Schwaiger M, Chang C-W, et al. Global reorganization of replication domains during embryonic stem cell differentiation. PLoS Biology. 2008 Oct 7;6(10). doi:10.1371/journal.pbio.0060245 
369.	Pope BD, Ryba T, Dileep V, Yue F, Wu W, Denas O, et al. Topologically associating domains are stable units of replication-timing regulation. Nature. 2014 Nov 19;515(7527):402–5. doi:10.1038/nature13986 
370.	Guelen L, Pagie L, Brasset E, Meuleman W, Faza MB, Talhout W, et al. Domain organization of human chromosomes revealed by mapping of nuclear lamina interactions. Nature. 2008 May 7;453(7197):948–51. doi:10.1038/nature06947 
371.	Scaffidi P, Misteli T. Lamin A-dependent nuclear defects in human aging. Science. 2006 May 19;312(5776):1059–63. doi:10.1126/science.1127168 
372.	Buchwalter A, Hetzer MW. Nucleolar expansion and elevated protein translation in premature aging. Nature Communications. 2017 Aug 30;8(1). doi:10.1038/s41467-017-00322-z 
373.	Scaffidi P, Misteli T. Reversal of the cellular phenotype in the premature aging disease Hutchinson-Gilford Progeria syndrome. Nature Medicine. 2005 Mar 6;11(4):440–5. doi:10.1038/nm1204 
374.	Oberdoerffer P, Sinclair DA. The role of nuclear architecture in genomic instability and ageing. Nature Reviews Molecular Cell Biology. 2007 Sept;8(9):692–702. doi:10.1038/nrm2238 
375.	St. Laurent G, Hammell N, McCaffrey TA. A line-1 component to human aging: Do line elements exact a longevity cost for evolutionary advantage? Mechanisms of Ageing and Development. 2010 May;131(5):299–305. doi:10.1016/j.mad.2010.03.008 
376.	Biggs LC, Kim CS, Miroshnikova YA, Wickström SA. Mechanical forces in the skin: Roles in tissue architecture, stability, and function. Journal of Investigative Dermatology. 2020 Feb;140(2):284–90. doi:10.1016/j.jid.2019.06.137 
377.	Nava MM, Miroshnikova YA, Biggs LC, Whitefield DB, Metge F, Boucas J, et al. Heterochromatin-driven nuclear softening protects the genome against mechanical stress-induced damage. Cell. 2020 May;181(4). doi:10.1016/j.cell.2020.03.052 
378.	Lombardi ML, Jaalouk DE, Shanahan CM, Burke B, Roux KJ, Lammerding J. The interaction between Nesprins and Sun proteins at the nuclear envelope is critical for force transmission between the nucleus and cytoskeleton. Journal of Biological Chemistry. 2011 Jul;286(30):26743–53. doi:10.1074/jbc.m111.233700 
379.	Hsia C-R, Melters DP, Dalal Y. The force is strong with this epigenome: Chromatin structure and mechanobiology. 2023 Mar 8; doi:10.31219/osf.io/s2h9j 
380.	Uhler C, Shivashankar GV. Chromosome intermingling: Mechanical hotspots for genome regulation. Trends in Cell Biology. 2017 Nov;27(11):810–9. doi:10.1016/j.tcb.2017.06.005 
381.	King T, Butcher S, Zalewski L. Apocrita - high performance computing cluster for Queen Mary University of London [Internet]. Zenodo; 2024 [cited 2026 Jan 6]. Available from: https://zenodo.org/records/438045 
382.	Summan SK. Simran PhD Thesis [Internet]. Github QMUL; 2025 [cited 2025]. Available from: https://github.com/hmz042/Simran_PhD_Thesis 
383.	Suzuki K, Bose P, Leong-Quong RY, Fujita DJ, Riabowol K. Reap: A two minute cell fractionation method. BMC Research Notes. 2010 Nov 10;3(1). doi:10.1186/1756-0500-3-294
384.	Müller L. Protocol to determine the subcellular localization of protein interactions in murine keratinocytes. STAR Protocols. 2023 Jun;4(2):102309. doi:10.1016/j.xpro.2023.102309 
385.	Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with deseq2. 2014 Feb 19; doi:10.1101/002832 
386.	Welcome to string [Internet]. [cited 2026 Jan 6]. Available from: https://string-db.org/ 
387.	Grandi FC, Modi H, Kampman L, Corces MR. Chromatin Accessibility Profiling by ATAC-Seq. Nature Protocols. 2022 Apr 27;17(6):1518–52. doi:10.1038/s41596-022-00692-9 
388.	Buenrostro JD, Wu B, Litzenburger UM, Ruff D, Gonzales ML, Snyder MP, et al. Single-cell chromatin accessibility reveals principles of regulatory variation. Nature. 2015 Jun 17;523(7561):486–90. doi:10.1038/nature14590 
389.	Bentsen M, Goymann P, Schultheis H, Klee K, Petrova A, Wiegandt R, et al. Beyond accessibility: ATAC-seq footprinting unravels kinetics of transcription factor binding during zygotic genome activation. Nature Communications. 2019 Dec 9; doi:10.1101/869560 
390.	Henikoff S, Henikoff JG, Ahmad K, Paranal RM, Janssens DH, Russell ZR, et al. Epigenomic analysis of formalin-fixed paraffin-embedded samples by CUT&TAG. Nature Communications. 2023 Sept 22;14(1). doi:10.1038/s41467-023-41666-z 
391.	Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The sequence alignment/map format and SAMtools. Bioinformatics. 2009 Jun 8;25(16):2078–9. doi:10.1093/bioinformatics/btp352
392.	 Ramírez F, Ryan DP, Grüning B, Bhardwaj V, Kilpert F, Richter AS, et al. Deeptools2: A next generation web server for deep-sequencing data analysis. Nucleic Acids Research. 2016 Apr 13;44(W1). doi:10.1093/nar/gkw257 
393.	Zeller P, Yeung J, Viñas Gaza H, de Barbanson BA, Bhardwaj V, Florescu M, et al. Single-cell sortchic identifies hierarchical chromatin dynamics during hematopoiesis. Nature Genetics. 2022 Dec 20;55(2):333–45. doi:10.1038/s41588-022-01260-3 
394.	Zhu Q, Liu N, Orkin SH, Yuan G-C. CUT&RUNTools: A flexible pipeline for CUT&RUN processing and footprint analysis. Genome Biology. 2019 Sept 9;20(1). doi:10.1186/s13059-019-1802-4 
395.	Jianhong. Jianhong/HMMTBROADPEAK: Call broad peak by a modified baum_welch algorithm that supports multi-dimensional student’s T emission. [Internet]. 2021 [cited 2025 Dec 11]. Available from: https://github.com/jianhong/HMMtBroadPeak 
396.	Mayakonda A, Westermann F. Trackplot: A fast and lightweight R script for epigenomic enrichment plots. Bioinformatics Advances. 2024 Jan 1;4(1). doi:10.1093/bioadv/vbae031 
397.	Li S, Wan C, Zheng R, Fan J, Dong X, Meyer CA, et al. Cistrome-Go: A web server for functional enrichment analysis of transcription factor chip-seq peaks. Nucleic Acids Research. 2019 May 4;47(W1). doi:10.1093/nar/gkz332 
398.	Toolkit for Cistrome Data browser [Internet]. [cited 2025 Dec 11]. Available from: http://dbtoolkit.cistrome.org/ 
399.	Zheng R, Wan C, Mei S, Qin Q, Wu Q, Sun H, et al. Cistrome data browser: Expanded datasets and new tools for Gene Regulatory Analysis. Nucleic Acids Research. 2018 Nov 20;47(D1). doi:10.1093/nar/gky1094 
400.	Mei S, Qin Q, Wu Q, Sun H, Zheng R, Zang C, et al. Cistrome Data browser: A Data Portal for ChIP-Seq and chromatin accessibility data in human and mouse. Nucleic Acids Research. 2016 Oct 26;45(D1). doi:10.1093/nar/gkw983 
401.	Furman D, Auwerx J, Bulteau A-L, Church G, Couturaud V, Crabbe L, et al. Skin health and biological aging. Nature Aging. 2025 Jun 17;5(7):1195–206. doi:10.1038/s43587-025-00901-6 
402.	Langton AK, Halai P, Griffiths CEM, Sherratt MJ, Watson REB. The impact of intrinsic ageing on the protein composition of the dermal-epidermal junction. Mechanisms of Ageing and Development. 2016 Jun;156:14–6. doi:10.1016/j.mad.2016.03.006 
403.	Persa OD, Koester J, Niessen CM. Regulation of cell polarity and tissue architecture in epidermal aging and cancer. Journal of Investigative Dermatology. 2021 Apr;141(4):1017–23. doi:10.1016/j.jid.2020.12.012 
404.	Breitenbach JS, Rinnerthaler M, Trost A, Weber M, Klausegger A, Gruber C, et al. Transcriptome and ultrastructural changes in dystrophic epidermolysis bullosa resemble skin aging. Aging. 2015 Jun 14;7(6):389–411. doi:10.18632/aging.100755 
405.	Lai W-F, Wong W-T. Roles of the actin cytoskeleton in aging and age-associated diseases. Ageing Research Reviews. 2020 Mar;58:101021. doi:10.1016/j.arr.2020.101021 
406.	Lüke Y, Zaim H, Karakesisoglou I, Jaeger VM, Sellin L, Lu W, et al. Nesprin-2 Giant (nuance) maintains nuclear envelope architecture and composition in Skin. Journal of Cell Science. 2008 Jun 1;121(11):1887–98. doi:10.1242/jcs.019075 
407.	Cantwell H, Nurse P. A systematic genetic screen identifies essential factors involved in nuclear size control. PLOS Genetics. 2019 Feb 13;15(2). doi:10.1371/journal.pgen.1007929 
408.	Carley E, Stewart RK, Zieman A, Jalilian I, King DianeE, Zubek A, et al. The LINC complex transmits integrin-dependent tension to the nuclear lamina and represses epidermal differentiation. 2020 May 4; doi:10.1101/2020.05.03.075085 
409.	Lima JT, Ferreira JG. Mechanobiology of the nucleus during the G2-M transition. Nucleus. 2024 Mar 27;15(1). doi:10.1080/19491034.2024.2330947 
410.	Belaadi N, Pernet L, Aureille J, Chadeuf G, Rio M, Vaillant N, et al. SUN2 regulates mitotic duration in response to extracellular matrix rigidity. Proceedings of the National Academy of Sciences. 2022 Nov 2;119(45). doi:10.1073/pnas.2116167119 
411.	Thakar K, May CK, Rogers A, Carroll CW. Opposing roles for distinct LINC complexes in regulation of the small gtpase rhoa. Molecular Biology of the Cell. 2017 Jan;28(1):182–91. doi:10.1091/mbc.e16-06-0467 
412.	Krshnan L, Siu WS, Van de Weijer M, Hayward D, Guerrero EN, Gruneberg U, et al. Regulated degradation of the inner nuclear membrane protein SUN2 maintains nuclear envelope architecture and function. eLife. 2022 Nov 1;11. doi:10.7554/elife.81573 
413.	Lei K, Zhu X, Xu R, Shao C, Xu T, Zhuang Y, et al. Inner nuclear envelope proteins SUN1 and SUN2 play a prominent role in the DNA damage response. Current Biology. 2012 Sept;22(17):1609–15. doi:10.1016/j.cub.2012.06.043 
414.	Stewart RM, Zubek AE, Rosowski KA, Schreiner SM, Horsley V, King MC. Nuclear–cytoskeletal linkages facilitate cross talk between the nucleus and intercellular adhesions. Journal of Cell Biology. 2015 May 11;209(3):403–18. doi:10.1083/jcb.201502024 
415.	Hervé S, Scelfo A, Bersano Marchisio G, Grison M, Vaidžiulytė K, Dumont M, et al. Chromosome mis-segregation triggers cell cycle arrest through a mechanosensitive nuclear envelope checkpoint. Nature Cell Biology. 2025 Jan;27(1):73–86. doi:10.1038/s41556-024-01565-x 
416.	Kim PH, Kim JR, Tu Y, Jung H, Jeong JY, Tran AP, et al. Progerin forms an abnormal meshwork and has a dominant-negative effect on the nuclear lamina. Proceedings of the National Academy of Sciences. 2024 Jun 25;121(27). doi:10.1073/pnas.2406946121 
417.	Danielsson BE, Tieu KV, Bathula K, Armiger TJ, Vellala PS, Taylor RE, et al. Lamin microaggregates lead to altered mechanotransmission in progerin-expressing cells. Nucleus. 2020 Jan 1;11(1):194–204. doi:10.1080/19491034.2020.1802906 
418.	Cho S, Vashisth M, Abbas A, Majkut S, Vogel K, Xia Y, et al. Mechanosensing by the lamina protects against nuclear rupture, DNA damage, and cell cycle arrest. 2019 Mar 20; doi:10.1101/583179 
419.	Gilbert HT, Mallikarjun V, Dobre O, Jackson MR, Pedley R, Gilmore AP, et al. Nuclear decoupling is part of a rapid protein-level cellular response to high-intensity mechanical loading. Nature Communications. 2019 Sept 12;10(1). doi:10.1038/s41467-019-11923-1 
420.	Amiad Pavlov D, Unnikannan C, Lorber D, Bajpai G, Olender T, Stoops E, et al. The LINC complex inhibits excessive chromatin repression. Cells. 2023 Mar 18;12(6):932. doi:10.3390/cells12060932 
421.	Sun W-W, Jiao S, Sun L, Zhou Z, Jin X, Wang J-H. SUN2 modulates HIV-1 infection and latency through association with lamin A/C to maintain the repressive chromatin. mBio. 2018 Jul 5;9(3). doi:10.1128/mbio.02408-17 
422.	Chang L, Li M, Shao S, Li C, Ai S, Xue B, et al. Nuclear peripheral chromatin-lamin B1 interaction is required for global integrity of chromatin architecture and dynamics in human cells. Protein &amp; Cell. 2020 Nov 5;13(4):258–80. doi:10.1007/s13238-020-00794-8 
423.	Murray-Nerger LA, Justice JL, Rekapalli P, Hutton JE, Cristea IM. Lamin B1 acetylation slows the G1 to S cell cycle transition through inhibition of DNA repair. Nucleic Acids Research. 2021 Feb 3;49(4):2044–64. doi:10.1093/nar/gkab019 
424.	Makrantonaki E, Brink TC, Zampeli V, Elewa RM, Mlody B, Hossini AM, et al. Identification of biomarkers of human skin ageing in both genders. wnt signalling - A label of skin ageing? PLoS ONE. 2012 Nov 30;7(11). doi:10.1371/journal.pone.0050393 
425.	Watanabe M, Natsuga K, Nishie W, Kobayashi Y, Donati G, Suzuki S, et al. Type XVII collagen coordinates proliferation in the interfollicular epidermis. eLife. 2017 Jul 11;6. doi:10.7554/elife.26635 
426.	Kaur A, Webster MR, Weeraratna AT. In the wnt-er of life: Wnt signalling in melanoma and ageing. British Journal of Cancer. 2016 Oct 20;115(11):1273–9. doi:10.1038/bjc.2016.332 
427.	Quan T, Fisher GJ. Role of age-associated alterations of the dermal extracellular matrix microenvironment in human skin aging: A mini-review. Gerontology. 2015;61(5):427–34. doi:10.1159/000371708 
428.	Fisher GJ, Shao Y, He T, Qin Z, Perry D, Voorhees JJ, et al. Reduction of fibroblast size/mechanical force down‐regulates            tgf            ‐β type            ii            receptor: Implications for human skin aging. Aging Cell. 2015 Oct 8;15(1):67–76. doi:10.1111/acel.12410 
429.	Qin Z, Fisher GJ, Voorhees JJ, Quan T. Actin cytoskeleton assembly regulates collagen production via TGF‐β type II receptor in human skin fibroblasts. Journal of Cellular and Molecular Medicine. 2018 Jun 11;22(9):4085–96. doi:10.1111/jcmm.13685 
430.	Han K-H, Choi H-R, Won C-H, Chung J-H, Cho K-H, Eun H-C, et al. Alteration of the TGF-β/SMAD pathway in intrinsically and UV-induced skin aging. Mechanisms of Ageing and Development. 2005 May;126(5):560–7. doi:10.1016/j.mad.2004.11.006 
431.	Compton C, Tong Y, Trookman N, Zhao H, Roy D. TGF-β1 gene expression in cultured human keratinocytes does not decrease with biologic age. Journal of Investigative Dermatology. 1994 Jul;103(1):127–33. doi:10.1111/1523-1747.ep12391899 
432.	Tanimura S, Tadokoro Y, Inomata K, Binh NT, Nishie W, Yamazaki S, et al. Hair follicle stem cells provide a functional niche for melanocyte stem cells. Cell Stem Cell. 2011 Feb;8(2):177–87. doi:10.1016/j.stem.2010.11.029 
433.	Martins VL, Caley MP, Moore K, Szentpetery Z, Marsh ST, Murrell DF, et al. Suppression of TGFβand angiogenesis by type VII collagen in cutaneous SCC. Journal of the National Cancer Institute. 2015 Oct 16;108(1). doi:10.1093/jnci/djv293 
434.	Chacón‐Solano E, León C, Díaz F, García‐García F, García M, Escámez MJ, et al. Fibroblast activation and abnormal extracellular matrix remodelling as common hallmarks in three cancer‐prone genodermatoses. British Journal of Dermatology. 2019 Apr 15;181(3):512–22. doi:10.1111/bjd.17698 
435.	Chi, Y.-H. et al. (2022) ‘Deformation of the nucleus by tgfβ1 via the remodeling of nuclear envelope and histone isoforms’, Epigenetics &amp; Chromatin, 15(1). doi:10.1186/s13072-021-00434-3. 
436.	Hutchison, C.J. (2014) ‘B-type lamins in Health and Disease’, Seminars in Cell &amp; Developmental Biology, 29, pp. 158–163. doi:10.1016/j.semcdb.2013.12.012. 
437.	Shah, P.P. et al. (2013) ‘Lamin B1 depletion in senescent cells triggers large-scale changes in gene expression and the chromatin landscape’, Genes &amp; Development, 27(16), pp. 1787–1799. doi:10.1101/gad.223834.113
438.	Pujadas Liwag, E.M. et al. (2024) ‘Depletion of lamins B1 and B2 promotes chromatin mobility and induces differential gene expression by a mesoscale-motion-dependent mechanism’, Genome Biology, 25(1). doi:10.1186/s13059-024-03212-y
439.	Buchwalter, A. and Hetzer, M.W. (2017) ‘Nucleolar expansion and elevated protein translation in premature aging’, Nature Communications, 8(1). doi:10.1038/s41467-017-00322-z
440.	Phan, T., Khalid, F. and Iben, S. (2019) ‘Nucleolar and ribosomal dysfunction—a common pathomechanism in childhood progerias?’, Cells, 8(6), p. 534. doi:10.3390/cells8060534
441.	Kang, S., Chovatiya, G. and Tumbar, T. (2019) ‘Epigenetic control in skin development, homeostasis and injury repair’, Experimental Dermatology, 28(4), pp. 453–463. doi:10.1111/exd.13872
442.	Biggs, L.C. et al. (2020) ‘Mechanical forces in the skin: Roles in tissue architecture, stability, and function’, Journal of Investigative Dermatology, 140(2), pp. 284–290. doi:10.1016/j.jid.2019.06.137
443.	Li, H. et al. (2021) ‘Nonlinear elasticity of biological basement membrane revealed by rapid inflation and deflation’, Proceedings of the National Academy of Sciences, 118(11). doi:10.1073/pnas.2022422118
444.	Lammerding, J. and Hsia, C.-R. (2020) ‘Feeling stressed? piezo1-mediated loss of heterochromatin buys time for long-term adaptation’, Cell, 181(4), pp. 760–762. doi:10.1016/j.cell.2020.04.041
445.	Cho, S., Irianto, J. and Discher, D.E. (2017) ‘Mechanosensing by the nucleus: From pathways to scaling relationships’, Journal of Cell Biology, 216(2), pp. 305–315. doi:10.1083/jcb.201610042
446.	Le, H.Q. et al. (2016b) ‘Mechanical regulation of transcription controls polycomb-mediated gene silencing during lineage commitment’, Nature Cell Biology, 18(8), pp. 864–875. doi:10.1038/ncb3387
447.	Miralles, F. et al. (2003) ‘Actin dynamics control SRF activity by regulation of its coactivator mal’, Cell, 113(3), pp. 329–342. doi:10.1016/s0092-8674(03)00278-2
448.	Hoffman, L.M. et al. (2020) ‘Mechanical stress triggers nuclear remodeling and the formation of transmembrane actin nuclear lines with associated nuclear pore complexes’, Molecular Biology of the Cell, 31(16), pp. 1774–1787. doi:10.1091/mbc.e19-01-0027
449.	Davidson, P.M. and Cadot, B. (2021) ‘Actin on and around the nucleus’, Trends in Cell Biology, 31(3), pp. 211–223. doi:10.1016/j.tcb.2020.11.009
450.	Itano, N. et al. (2003) ‘Cell spreading controls endoplasmic and nuclear calcium: A physical gene regulation pathway from the cell surface to the nucleus’, Proceedings of the National Academy of Sciences, 100(9), pp. 5181–5186. doi:10.1073/pnas.0531397100
451.	Stephens, A.D. et al. (2017) ‘Chromatin and lamin a determine two different mechanical response regimes of the cell nucleus’, Molecular Biology of the Cell, 28(14), pp. 1984–1996. doi:10.1091/mbc.e16-09-0653
452.	Nava, M.M. et al. (2020) ‘Heterochromatin-driven nuclear softening protects the genome against mechanical stress-induced damage’, Cell, 181(4). doi:10.1016/j.cell.2020.03.052
453.	Le, H.Q. et al. (2016a) ‘Mechanical regulation of transcription controls polycomb-mediated gene silencing during lineage commitment’, Nature Cell Biology, 18(8), pp. 864–875. doi:10.1038/ncb3387
454.	Wales, P. et al. (2016) ‘Calcium-mediated actin reset (CAAR) mediates acute cell adaptations’, eLife, 5. doi:10.7554/elife.19850. 
455.	Shim H, Yoon J, Lee H, Karunasagara S, Li CJ, Shin Y, et al. Temporal stretch‐induced nuclear mechanosensing coordinates early chromatin accessibility and genome protection. Advanced Science. 2025 Oct 7; doi:10.1002/advs.202510554 
456.	Dupont S, Wickström SA. Mechanical regulation of chromatin and transcription. Nature Reviews Genetics. 2022 May 23;23(10):624–43. doi:10.1038/s41576-022-00493-6 
457.	Elosegui-Artola A, Andreu I, Beedle AEM, Lezamiz A, Uroz M, Kosmalska AJ, et al. Force triggers yap nuclear entry by regulating transport across nuclear pores. Cell. 2017 Nov;171(6). doi:10.1016/j.cell.2017.10.008 
458.	Sen B, Xie Z, Howard S, Styner M, van Wijnen AJ, Uzer G, et al. Mechanically induced nuclear shuttling of β-catenin requires co-transfer of actin. Stem Cells. 2022 Feb 8;40(4):423–34. doi:10.1093/stmcls/sxac006 
459.	Kofler M, Kapus A. Nuclear import and export of yap and taz. Cancers. 2023 Oct 12;15(20):4956. doi:10.3390/cancers15204956 
460.	Killaars AR, Grim JC, Walker CJ, Hushka EA, Brown TE, Anseth KS. Extended exposure to stiff microenvironments leads to persistent chromatin remodeling in human mesenchymal stem cells. Advanced Science. 2018 Dec 10;6(3). doi:10.1002/advs.201801483 
461.	Nasrollahi S, Walter C, Loza AJ, Schimizzi GV, Longmore GD, Pathak A. Past matrix stiffness primes epithelial cells and regulates their future collective migration through a mechanical memory. Biomaterials. 2017 Nov;146:146–55. doi:10.1016/j.biomaterials.2017.09.012 
462.	Sladitschek-Martens HL, Guarnieri A, Brumana G, Zanconato F, Battilana G, Xiccato RL, et al. Yap/TAZ activity in stromal cells prevents ageing by controlling cgas–sting. Nature. 2022 Jun 29;607(7920):790–8. doi:10.1038/s41586-022-04924-6 
463.	Kechagia Z, Sáez P, Gómez-González M, Canales B, Viswanadha S, Zamarbide M, et al. The laminin–keratin link shields the nucleus from mechanical deformation and signalling. Nature Materials. 2023 Sept 14;22(11):1409–20. doi:10.1038/s41563-023-01657-3 
464.	Bertillot F, Miroshnikova YA, Wickström SA. Snapshot: Mechanotransduction in the nucleus. Cell. 2022 Sept;185(19). doi:10.1016/j.cell.2022.08.017 
465.	Miroshnikova YA, Wickström SA. Mechanical forces in nuclear organization. Cold Spring Harbor Perspectives in Biology. 2021 Jun 29;14(1). doi:10.1101/cshperspect.a039685 
466.	Dahl KN, Kahn SM, Wilson KL, Discher DE. The nuclear envelope lamina network has elasticity and a compressibility limit suggestive of a molecular shock absorber. Journal of Cell Science. 2004 Sept 15;117(20):4779–86. doi:10.1242/jcs.01357 
467.	Dahl KN, Engler AJ, Pajerowski JD, Discher DE. Power-law rheology of isolated nuclei with deformation mapping of nuclear substructures. Biophysical Journal. 2005 Oct;89(4):2855–64. doi:10.1529/biophysj.105.062554 
468.	Lomakin AJ, Cattin CJ, Cuvelier D, Alraies Z, Molina M, Nader GP, et al. The nucleus acts as a ruler tailoring cell responses to spatial constraints. Science. 2020 Oct 16;370(6514). doi:10.1126/science.aba2894 
469.	Swift J, Ivanovska IL, Buxboim A, Harada T, Dingal PC, Pinter J, et al. Nuclear lamin-a scales with tissue stiffness and enhances matrix-directed differentiation. Science. 2013 Aug 30;341(6149). doi:10.1126/science.1240104 
470.	Swift J, Discher DE. The nuclear lamina is mechano-responsive to ECM elasticity in mature tissue. Journal of Cell Science. 2014 Jan 1; doi:10.1242/jcs.149203 
471.	Chung H, Oh S, Shin H-W, Lee Y, Lee H, Seok SH. Matrix stiffening enhances DNCB-induced IL-6 secretion in keratinocytes through activation of ERK and PI3K/akt pathway. Frontiers in Immunology. 2021 Nov 11;12. doi:10.3389/fimmu.2021.759992 
472.	Pundel OJ, Blowes LM, Connelly JT. Extracellular adhesive cues physically define nucleolar structure and function. Advanced Science. 2022 Feb 5;9(10). doi:10.1002/advs.202105545 
473.	De Magistris P, Antonin W. The dynamic nature of the nuclear envelope. Current Biology. 2018 Apr;28(8). doi:10.1016/j.cub.2018.01.073 
474.	Janssen AF, Breusegem SY, Larrieu D. Current methods and pipelines for image-based quantitation of nuclear shape and nuclear envelope abnormalities. Cells. 2022 Jan 20;11(3):347. doi:10.3390/cells11030347 
475.	Espada J, Varela I, Flores I, Ugalde AP, Cadiñanos J, Pendás AM, et al. Nuclear envelope defects cause stem cell dysfunction in premature-aging mice. The Journal of Cell Biology. 2008 Mar 31;181(1):27–35. doi:10.1083/jcb.200801096 
476.	Pathak RU, Soujanya M, Mishra RK. Deterioration of nuclear morphology and architecture: A hallmark of senescence and aging. Ageing Research Reviews. 2021 May;67:101264. doi:10.1016/j.arr.2021.101264 
477.	Lammerding J, Fong LG, Ji JY, Reue K, Stewart CL, Young SG, et al. Lamins A and C but not lamin B1 Regulate Nuclear Mechanics. Journal of Biological Chemistry. 2006 Sept;281(35):25768–80. doi:10.1074/jbc.m513511200 
478.	Guilluy C, Osborne LD, Van Landeghem L, Sharek L, Superfine R, Garcia-Mata R, et al. Isolated nuclei adapt to force and reveal a mechanotransduction pathway in the nucleus. Nature Cell Biology. 2014 Mar 9;16(4):376–81. doi:10.1038/ncb2927 
479.	Ho CY, Jaalouk DE, Vartiainen MK, Lammerding J. Lamin A/C and Emerin regulate MKL1–SRF activity by modulating actin dynamics. Nature. 2013 May 5;497(7450):507–11. doi:10.1038/nature12105 
480.	Essawy N, Samson C, Petitalot A, Moog S, Bigot A, Herrada I, et al. An emerin lem-domain mutation impairs cell response to mechanical stress. Cells. 2019 Jun 10;8(6):570. doi:10.3390/cells8060570 
481.	Holaska JM, Kowalski AK, Wilson KL. Emerin caps the pointed end of actin filaments: Evidence for an actin cortical network at the nuclear inner membrane. PLoS Biology. 2004 Aug 24;2(9). doi:10.1371/journal.pbio.0020231 
482.	Spencer VA, Costes S, Inman JL, Xu R, Chen J, Hendzel MJ, et al. Depletion of nuclear actin is a key mediator of quiescence in epithelial cells. Journal of Cell Science. 2011 Jan 1;124(1):123–32. doi:10.1242/jcs.073197 
483.	Stephens AD, Liu PZ, Banigan EJ, Almassalha LM, Backman V, Adam SA, et al. Chromatin histone modifications and rigidity affect nuclear morphology independent of lamins. Molecular Biology of the Cell. 2018 Jan 15;29(2):220–33. doi:10.1091/mbc.e17-06-0410 
484.	Stephens AD, Liu PZ, Banigan EJ, Almassalha LM, Backman V, Adam SA, et al. Chromatin histone modifications and rigidity affect nuclear morphology independent of lamins. Molecular Biology of the Cell. 2018 Jan 15;29(2):220–33. doi:10.1091/mbc.e17-06-0410 
485.	Ghosh S, Seelbinder B, Henderson JT, Watts RD, Scott AK, Veress AI, et al. Deformation microscopy for dynamic intracellular and intranuclear mapping of mechanics with high spatiotemporal resolution. Cell Reports. 2019 Apr;27(5). doi:10.1016/j.celrep.2019.04.009 
486.	Schreiner SM, Koo PK, Zhao Y, Mochrie SG, King MC. The tethering of chromatin to the nuclear envelope supports nuclear mechanics. Nature Communications. 2015 Jun 15;6(1). doi:10.1038/ncomms8159 
487.	Stephens AD, Liu PZ, Kandula V, Chen H, Almassalha LM, Herman C, et al. Physicochemical mechanotransduction alters nuclear shape and mechanics via Heterochromatin Formation. Molecular Biology of the Cell. 2019 Jun 19; doi:10.1091/mbc.e19-05-0286-t 
488.	Hobson CM, Kern M, O’Brien ET, Stephens AD, Falvo MR, Superfine R. Correlating nuclear morphology and external force with combined atomic force microscopy and light sheet imaging separates roles of chromatin and lamin A/C in nuclear mechanics. Molecular Biology of the Cell. 2020 Jul 21;31(16):1788–801. doi:10.1091/mbc.e20-01-0073 
489.	Nava MM, Miroshnikova YA, Biggs LC, Whitefield DB, Metge F, Boucas J, et al. Heterochromatin-driven nuclear softening protects the genome against mechanical stress-induced damage. Cell. 2020 May;181(4). doi:10.1016/j.cell.2020.03.052 
490.	Le HQ, Ghatak S, Yeung C-YC, Tellkamp F, Günschmann C, Dieterich C, et al. Mechanical regulation of transcription controls polycomb-mediated gene silencing during lineage commitment. Nature Cell Biology. 2016 Jul 11;18(8):864–75. doi:10.1038/ncb3387 
491.	Yue X, Cui J, Sun Z, Liu L, Li Y, Shao L, et al. Nuclear softening mediated by SUN2 suppression delays mechanical stress-induced cellular senescence. Cell Death Discovery. 2023 May 17;9(1). doi:10.1038/s41420-023-01467-1 
492.	Vining KH, Mooney DJ. Mechanical forces direct stem cell behaviour in development and regeneration. Nature Reviews Molecular Cell Biology. 2017 Nov 8;18(12):728–42. doi:10.1038/nrm.2017.108 
493.	Evans ND, Oreffo ROC, Healy E, Thurner PJ, Man YH. Epithelial mechanobiology, skin wound healing, and the stem cell niche. Journal of the Mechanical Behavior of Biomedical Materials. 2013 Dec;28:397–409. doi:10.1016/j.jmbbm.2013.04.023 
494.	Püllen R, Konrad J, Merkel R, Hoffmann B. Skin under strain: From epithelial model tissues to adult epithelia. Cells. 2021 Jul 20;10(7):1834. doi:10.3390/cells10071834 
495.	Perlman CE, Bhattacharya J. Alveolar expansion imaged by optical sectioning microscopy. Journal of Applied Physiology. 2007 Sept;103(3):1037–44. doi:10.1152/japplphysiol.00160.2007 
496.	Blanchard GB, Kabla AJ, Schultz NL, Butler LC, Sanson B, Gorfinkiel N, et al. Tissue tectonics: Morphogenetic strain rates, cell shape change and intercalation. Nature Methods. 2009 May 3;6(6):458–64. doi:10.1038/nmeth.1327 
497.	Kim E, Riehl BD, Bouzid T, Yang R, Duan B, Donahue HJ, et al. Yap mechanotransduction under cyclic mechanical stretch loading for mesenchymal stem cell osteogenesis is regulated by rock. Frontiers in Bioengineering and Biotechnology. 2024 Jan 11;11. doi:10.3389/fbioe.2023.1306002 
498.	Discher DE, Janmey P, Wang Y. Tissue cells feel and respond to the stiffness of their substrate. Science. 2005 Nov 18;310(5751):1139–43. doi:10.1126/science.1116995 
499.	Kim J-K, Louhghalam A, Lee G, Schafer BW, Wirtz D, Kim D-H. Nuclear lamin A/C harnesses the perinuclear apical actin cables to protect nuclear morphology. Nature Communications. 2017 Dec 14;8(1). doi:10.1038/s41467-017-02217-5 
500.	Yano S, Komine M, Fujimoto M, Okochi H, Tamaki K. Mechanical stretching in vitro regulates signal transduction pathways and cellular proliferation in human epidermal keratinocytes. Journal of Investigative Dermatology. 2004 Mar;122(3):783–90. doi:10.1111/j.0022-202x.2004.22328.x 
501.	Flick Jaecker F, Almeida JA, Krull CM, Pathak A. Nucleoli in epithelial cell collectives respond to tumorigenic, spatial, and mechanical cues. Molecular Biology of the Cell. 2022 Sept 15;33(11). doi:10.1091/mbc.e22-02-0070 
502.	Pundel OJ, Blowes LM, Connelly JT. Extracellular adhesive cues physically define nucleolar structure and function. Advanced Science. 2022 Feb 5;9(10). doi:10.1002/advs.202105545 
503.	Iyer KV, Taubenberger A, Zeidan SA, Dye NA, Eaton S, Jülicher F. Apico-basal cell compression regulates lamin A/C levels in epithelial tissues. Nature Communications. 2021 Mar 25;12(1). doi:10.1038/s41467-021-22010-9 
504.	Rogerson C, Wotherspoon DJ, Tommasi C, Button RW, O’Shaughnessy RF. Akt1-associated actomyosin remodelling is required for nuclear lamina dispersal and nuclear shrinkage in epidermal terminal differentiation. Cell Death &amp; Differentiation. 2021 Jan 18;28(6):1849–64. doi:10.1038/s41418-020-00712-9 
505.	Gilbert HT, Mallikarjun V, Dobre O, Jackson MR, Pedley R, Gilmore AP, et al. Nuclear decoupling is part of a rapid protein-level cellular response to high-intensity mechanical loading. Nature Communications. 2019 Sept 12;10(1). doi:10.1038/s41467-019-11923-1 
506.	Lynch B, Pageon H, Le Blay H, Brizion S, Bastien P, Bornschlögl T, et al. A mechanistic view on the aging human skin through ex vivo layer-by-layer analysis of mechanics and microstructure of facial and mammary dermis. Scientific Reports. 2022 Jan 17;12(1). doi:10.1038/s41598-022-04767-1 
507.	Maurer M, Lammerding J. The driving force: Nuclear mechanotransduction in cellular function, fate, and disease. Annual Review of Biomedical Engineering. 2019 Jun 4;21(1):443–68. doi:10.1146/annurev-bioeng-060418-052139 
508.	de Lima Camillo LP, Asif MH, Horvath S, Larschan E, Singh R. Histone mark age of human tissues and cell types. Science Advances. 2025 Jan 3;11(1). doi:10.1126/sciadv.adk9373 
509.	Zhang T, Cooper S, Brockdorff N. The interplay of histone modifications – writers that read. EMBO reports. 2015 Oct 16;16(11):1467–81. doi:10.15252/embr.201540945 
510.	Kimura H. Histone modifications for human epigenome analysis. Journal of Human Genetics. 2013 Jun 6;58(7):439–45. doi:10.1038/jhg.2013.66 
511.	Pal D, Patel M, Boulet F, Sundarraj J, Grant OA, Branco MR, et al. H4K16AC activates the transcription of transposable elements and contributes to their cis-regulatory function. Nature Structural &amp; Molecular Biology. 2023 Jun 12;30(7):935–47. doi:10.1038/s41594-023-01016-5 
512.	Wang Z, Schones DE, Zhao K. Characterization of human epigenomes. Current Opinion in Genetics &amp; Development. 2009 Apr;19(2):127–34. doi:10.1016/j.gde.2009.02.001 
513.	Botchkarev VA, Gdula MR, Mardaryev AN, Sharov AA, Fessing MY. Epigenetic regulation of gene expression in keratinocytes. Journal of Investigative Dermatology. 2012 Nov;132(11):2505–21. doi:10.1038/jid.2012.182 
514.	Solé‐Boldo L, Raddatz G, Gutekunst J, Gilliam O, Bormann F, Liberio MS, et al. Differentiation‐related epigenomic changes define clinically distinct keratinocyte cancer subclasses. Molecular Systems Biology. 2022 Sept;18(9). doi:10.15252/msb.202211073 
515.	Bou Sleiman M, Jha P, Houtkooper R, Williams RW, Wang X, Auwerx J. The gene-regulatory footprint of aging highlights conserved central regulators. Cell Reports. 2020 Sept;32(13):108203. doi:10.1016/j.celrep.2020.108203 
516.	Lien W-H, Guo X, Polak L, Lawton LN, Young RA, Zheng D, et al. Genome-wide maps of histone modifications unwind in vivo chromatin states of the hair follicle lineage. Cell Stem Cell. 2011 Sept;9(3):219–32. doi:10.1016/j.stem.2011.07.015 
517.	Egolf S, Aubert Y, Doepner M, Anderson A, Maldonado-Lopez A, Pacella G, et al. LSD1 inhibition promotes epithelial differentiation through derepression of fate-determining transcription factors. Cell Reports. 2019 Aug;28(8). doi:10.1016/j.celrep.2019.07.058 
518.	Lee B, Villarreal-Ponce A, Fallahi M, Ovadia J, Sun P, Yu Q-C, et al. Transcriptional mechanisms link epithelial plasticity to adhesion and differentiation of epidermal progenitor cells. Developmental Cell. 2014 Apr;29(1):47–58. doi:10.1016/j.devcel.2014.03.005 
519.	Mlacki M, Darido C, Jane SM, Wilanowski T. Loss of grainy head-like 1 is associated with disruption of the epidermal barrier and squamous cell carcinoma of the skin. PLoS ONE. 2014 Feb 20;9(2). doi:10.1371/journal.pone.0089247 
520.	Sen GL, Boxer LD, Webster DE, Bussat RT, Qu K, Zarnegar BJ, et al. ZNF750 is a P63 target gene that induces KLF4 to drive terminal epidermal differentiation. Developmental Cell. 2012 Mar;22(3):669–77. doi:10.1016/j.devcel.2011.12.001 
521.	Bernstein BE, Mikkelsen TS, Xie X, Kamal M, Huebert DJ, Cuff J, et al. A bivalent chromatin structure marks key developmental genes in embryonic stem cells. Cell. 2006 Apr;125(2):315–26. doi:10.1016/j.cell.2006.02.041 
522.	Azuara V, Perry P, Sauer S, Spivakov M, Jørgensen HF, John RM, et al. Chromatin signatures of pluripotent cell lines. Nature Cell Biology. 2006 Mar 29;8(5):532–8. doi:10.1038/ncb1403 
523.	Shah PP, Donahue G, Otte GL, Capell BC, Nelson DM, Cao K, et al. Lamin B1 depletion in senescent cells triggers large-scale changes in gene expression and the chromatin landscape. Genes &amp; Development. 2013 Aug 9;27(16):1787–99. doi:10.1101/gad.223834.113 
524.	Heinz S, Romanoski CE, Benner C, Glass CK. The selection and function of cell type-specific enhancers. Nature Reviews Molecular Cell Biology. 2015 Feb 4;16(3):144–54. doi:10.1038/nrm3949 
525.	Yao Q, Epstein CB, Banskota S, Issner R, Kim Y, Bernstein BE, et al. Epigenetic alterations in keratinocyte carcinoma. Journal of Investigative Dermatology. 2021 May;141(5):1207–18. doi:10.1016/j.jid.2020.10.018 
526.	Yang JH, Hayano M, Griffin PT, et al. Loss of epigenetic information as a cause of mammalian aging. Cell. 2023;186(2):305-326.e27. DOI: 10.1016/j.cell.2022.12.027. 
527.	Milan M, Runfola V, Patel M, Falbo L, Noberini R, Soriani C, et al. Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression. Nucleic Acids Research. 2025 Sept 23;53(18). doi:10.1093/nar/gkaf916 
528.	Monserrat J, Morales Torres C, Richardson L, Wilson TS, Patel H, Domart M-C, et al. Disruption of the MSL complex inhibits tumour maintenance by exacerbating chromosomal instability. Nature Cell Biology. 2021 Apr;23(4):401–12. doi:10.1038/s41556-021-00657-2 
529.	Pal D, Patel M, Boulet F, Sundarraj J, Grant OA, Branco MR, et al. H4K16ac activates the transcription of transposable elements and contributes to their cis-regulatory function. Nature Structural &amp; Molecular Biology. 2023 Jun 12;30(7):935–47. doi:10.1038/s41594-023-01016-5 
530.	Karoutas A, Szymanski W, Rausch T, Guhathakurta S, Rog-Zielinska EA, Peyronnet R, et al. The NSL complex maintains nuclear architecture stability via lamin A/C acetylation. Nature Cell Biology. 2019 Oct;21(10):1248–60. doi:10.1038/s41556-019-0397-z 
531.	Taipale M, Rea S, Richter K, Vilar A, Lichter P, Imhof A, et al. HMOF histone acetyltransferase is required for histone H4 lysine 16 acetylation in mammalian cells. Molecular and Cellular Biology. 2005 Aug 1;25(15):6798–810. doi:10.1128/mcb.25.15.6798-6810.2005 
532.	Dube CT, Jahan FR, Lim CY. Key changes in chromatin mark mammalian epidermal differentiation and ageing. Epigenetics. 2021 Apr 23;17(4):444–59. doi:10.1080/15592294.2021.1917812 
533.	Klemm SL, Shipony Z, Greenleaf WJ. Chromatin accessibility and the regulatory epigenome. Nature Reviews Genetics. 2019 Jan 23;20(4):207–20. doi:10.1038/s41576-018-0089-8
534.	Zhang C, Wang D, Dowell R, Yi R. Single cell analysis of transcriptome and open chromatin reveals the dynamics of hair follicle stem cell aging. Frontiers in Aging. 2023 Jul 3;4. doi:10.3389/fragi.2023.1192149 
535.	Qu J, Yi G, Zhou H. P63 cooperates with CTCF to modulate chromatin architecture in skin keratinocytes. Epigenetics &amp; Chromatin. 2019 Jun 4;12(1). doi:10.1186/s13072-019-0280-y 
536.	Corpet A, Stucki M. Chromatin maintenance and dynamics in senescence: A spotlight on Sahf Formation and the epigenome of Senescent Cells. Chromosoma. 2014 May 27;123(5):423–36. doi:10.1007/s00412-014-0469-6 
537.	Narita M, Nuñez S, Heard E, Narita M, Lin AW, Hearn SA, et al. RB-mediated heterochromatin formation and silencing of E2F target genes during cellular senescence. Cell. 2003 Jun;113(6):703–16. doi:10.1016/s0092-8674(03)00401-x 
538.	Juan AH, Wang S, Ko KD, Zare H, Tsai P-F, Feng X, et al. Roles of h3k27me2 and h3k27me3 examined during fate specification of embryonic stem cells. Cell Reports. 2017 Jan;18(1):297. doi:10.1016/j.celrep.2016.12.036 
539.	Sen GL, Webster DE, Barragan DI, Chang HY, Khavari PA. Control of differentiation in a self-renewing mammalian tissue by the histone demethylase JMJD3. Genes &amp; Development. 2008 Jul 15;22(14):1865–70. doi:10.1101/gad.1673508 
540.	Shumaker DK, Dechat T, Kohlmaier A, Adam SA, Bozovsky MR, Erdos MR, et al. Mutant Nuclear Lamin A leads to progressive alterations of epigenetic control in premature aging. Proceedings of the National Academy of Sciences. 2006 Jun 6;103(23):8703–8. doi:10.1073/pnas.0602569103 
541.	Cordisco S, Maurelli R, Bondanza S, Stefanini M, Zambruno G, Guerra L, et al. BMI-1 reduction plays a key role in physiological and premature aging of primary human keratinocytes. Journal of Investigative Dermatology. 2010 Apr;130(4):1048–62. doi:10.1038/jid.2009.355 
542.	McCord RP, Nazario-Toole A, Zhang H, Chines PS, Zhan Y, Erdos MR, et al. Correlated alterations in genome organization, histone methylation, and DNA–lamin A/C interactions in hutchinson-gilford progeria syndrome. Genome Research. 2012 Nov 14;23(2):260–9. doi:10.1101/gr.138032.112 
543.	Wang J, Zhang Y, Gao Y, Shan S, Li Q. EZH2 regulates the correlation between skin regeneration and the duration of mechanical stretch. Journal of Investigative Dermatology. 2021 Apr;141(4). doi:10.1016/j.jid.2020.09.007 
544.	Zhou VW, Goren A, Bernstein BE. Charting histone modifications and the functional organization of mammalian genomes. Nature Reviews Genetics. 2010 Nov 30;12(1):7–18. doi:10.1038/nrg2905 
545.	Verrecchia F, Pessah M, Atfi A, Mauviel A. Tumor necrosis factor-α inhibits transforming growth factor-β /SMAD signaling in human dermal fibroblasts via AP-1 activation. Journal of Biological Chemistry. 2000 Sept;275(39):30226–31. doi:10.1074/jbc.m005310200 
546.	Purohit T, He T, Qin Z, Li T, Fisher GJ, Yan Y, et al. Smad3-dependent regulation of type I collagen in human dermal fibroblasts: Impact on human skin connective tissue aging. Journal of Dermatological Science. 2016 Jul;83(1):80–3. doi:10.1016/j.jdermsci.2016.04.004 
547.	Naharro-Rodriguez J, Bacci S, Hernandez-Bule ML, Perez-Gonzalez A, Fernandez-Guarino M. Decoding skin aging: A review of mechanisms, markers, and modern therapies. Cosmetics. 2025 Jul 10;12(4):144. doi:10.3390/cosmetics12040144 
548.	Zou Z, Long X, Zhao Q, Zheng Y, Song M, Ma S, et al. A single-cell transcriptomic atlas of human skin aging. Developmental Cell. 2021 Feb;56(3). doi:10.1016/j.devcel.2020.11.002 
549.	García-García VA, Alameda JP, Page A, Casanova ML. Role of NF-ΚB in ageing and age-related diseases: Lessons from genetically modified mouse models. Cells. 2021 Jul 27;10(8):1906. doi:10.3390/cells10081906 
550.	Grabowska W, Sikora E, Bielak-Zmijewska A. Sirtuins, a promising target in slowing down the ageing process. Biogerontology. 2017 Mar 3;18(4):447–76. doi:10.1007/s10522-017-9685-9 
551.	Han K-H, Choi H-R, Won C-H, Chung J-H, Cho K-H, Eun H-C, et al. Alteration of the TGF-β/SMAD pathway in intrinsically and UV-induced skin aging. Mechanisms of Ageing and Development. 2005 May;126(5):560–7. doi:10.1016/j.mad.2004.11.006 
552.	Yeung F, Hoberg JE, Ramsey CS, Keller MD, Jones DR, Frye RA, et al. Modulation of nf‐κb‐dependent transcription and cell survival by the SIRT1 deacetylase. The EMBO Journal. 2004 May 20;23(12):2369–80. doi:10.1038/sj.emboj.7600244 
553.	Kawahara TLA, Michishita E, Adler AS, Damian M, Berber E, Lin M, et al. SIRT6 links histone H3 lysine 9 deacetylation to NF-ΚB-dependent gene expression and organismal life span. Cell. 2009 Jan;136(1):62–74. doi:10.1016/j.cell.2008.10.052 
554.	Mori R, Tanaka K, de Kerckhove M, Okamoto M, Kashiyama K, Tanaka K, et al. Reduced FOXO1 expression accelerates skin wound healing and attenuates scarring. The American Journal of Pathology. 2014 Sept;184(9):2465–79. doi:10.1016/j.ajpath.2014.05.012 
555.	Zhou W, Cao Q, Peng Y, Zhang Q, Castrillon DH, DePinho RA, et al. FOXO4 inhibits NF-ΚB and protects mice against colonic injury and inflammation. Gastroenterology. 2009 Oct;137(4):1403–14. doi:10.1053/j.gastro.2009.06.049 
556.	Li Y, Giovannini S, Wang T, Fang J, Li P, Shao C, et al. P63: A crucial player in epithelial stemness regulation. Oncogene. 2023 Oct 17;42(46):3371–84. doi:10.1038/s41388-023-02859-4 
557.	Koster MI, Dai D, Marinari B, Sano Y, Costanzo A, Karin M, et al. P63 induces key target genes required for epidermal morphogenesis. Proceedings of the National Academy of Sciences. 2007 Feb 27;104(9):3255–60. doi:10.1073/pnas.0611376104 
558.	Leonard MK, Kommagani R, Payal V, Mayo LD, Shamma HN, Kadakia MP. ΔNp63α regulates keratinocyte proliferation by controlling PTEN expression and localization. Cell Death &amp; Differentiation. 2011 Jun 3;18(12):1924–33. doi:10.1038/cdd.2011.73 
559.	Oh J-E, Kim RH, Shin K-H, Park N-H, Kang MK. ΔNp63α protein triggers epithelial-mesenchymal transition and confers stem cell properties in normal human keratinocytes. Journal of Biological Chemistry. 2011 Nov;286(44):38757–67. doi:10.1074/jbc.m111.244939 
560.	Di Girolamo D, Di Iorio E, Missero C. Molecular and cellular function of p63 in skin development and genetic diseases. Journal of Investigative Dermatology. 2025 Apr;145(4):766–79. doi:10.1016/j.jid.2024.08.011 
561.	Truong AB, Kretz M, Ridky TW, Kimmel R, Khavari PA. P63 regulates proliferation and differentiation of developmentally mature keratinocytes. Genes &amp; Development. 2006 Nov 15;20(22):3185–97. doi:10.1101/gad.1463206 
562.	Medawar A, Virolle T, Rostagno P, de la Forest-Divonne S, Gambaro K, Rouleau M, et al. ΔNp63 is essential for epidermal commitment of embryonic stem cells. PLoS ONE. 2008 Oct 17;3(10). doi:10.1371/journal.pone.0003441 
563.	Chin SS, Romano R-A, Nagarajan P, Sinha S, Garrett-Sinha LA. Aberrant epidermal differentiation and disrupted ΔNp63/notch regulatory axis in ETS1 transgenic mice. Biology Open. 2013 Oct 23;2(12):1336–45. doi:10.1242/bio.20135397 
564.	Yi M, Tan Y, Wang L, Cai J, Li X, Zeng Z, et al. TP63 links chromatin remodeling and enhancer reprogramming to epidermal differentiation and squamous cell carcinoma development. Cellular and Molecular Life Sciences. 2020 May 23;77(21):4325–46. doi:10.1007/s00018-020-03539-2 
565.	Yu X, Singh PK, Tabrejee S, Sinha S, Buck MJ. ΔNp63 is a pioneer factor that binds inaccessible chromatin and elicits chromatin remodeling. Epigenetics &amp; Chromatin. 2021 Apr 17;14(1). doi:10.1186/s13072-021-00394-8 
566.	Kudo K, Tsuyama N, Nagata K, Imaoka T, Iizuka D, Sugai-Takahashi M, et al. ΔNp63α transcriptionally represses p53 target genes involved in the radiation-induced DNA damage response. Radiation Oncology. 2022 Nov 15;17(1). doi:10.1186/s13014-022-02139-7 
567.	Romano R-A, Smalley K, Magraw C, Serna VA, Kurita T, Raghavan S, et al. Δnp63 knockout mice reveal its indispensable role as a master regulator of epithelial development and differentiation. Development. 2012 Feb 15;139(4):772–82. doi:10.1242/dev.071191 
568.	Zhu L, Rorke EA, Eckert RL. ΔNp63α promotes apoptosis of human epidermal keratinocytes. Journal of Investigative Dermatology. 2007 Aug;127(8):1980–91. doi:10.1038/sj.jid.5700797 
569.	Antonini D, Sirico A, Aberdam E, Ambrosio R, Campanile C, Fagoonee S, et al. A composite enhancer regulates P63 gene expression in epidermal morphogenesis and in keratinocyte differentiation by multiple mechanisms. Nucleic Acids Research. 2015 Jan 7;43(2):862–74. doi:10.1093/nar/gku1396 
570.	Kuang L, Li C. ΔNp63α-mediated epigenetic regulation in keratinocyte senescence. Epigenetics. 2023 Feb 9;18(1). doi:10.1080/15592294.2023.2173931 
571.	Panatta E, Lena AM, Mancini M, Affinati M, Smirnov A, Annicchiarico-Petruzzelli M, et al. Kruppel-like factor 4 regulates keratinocyte senescence. Biochemical and Biophysical Research Communications. 2018 May;499(2):389–95. doi:10.1016/j.bbrc.2018.03.172 
572.	Abbasova L, Urbanaviciute P, Hu D, Ismail JN, Schilder BM, Nott A, et al. CUT&TAG recovers up to half of encode chip-seq histone acetylation peaks. Nature Communications. 2025 Mar 27;16(1). doi:10.1038/s41467-025-58137-2 
573.	Henikoff S, Henikoff JG, Kaya-Okur HS, Ahmad K. Efficient chromatin accessibility mapping                   in situ                  by nucleosome-tethered tagmentation. 2020 Apr 17; doi:10.1101/2020.04.15.043083 
574.	Henikoff S, Henikoff JG, Ahmad K, Paranal RM, Janssens DH, Russell ZR, et al. Epigenomic analysis of formalin-fixed paraffin-embedded samples by CUT&TAG. Nature Communications. 2023 Sept 22;14(1). doi:10.1038/s41467-023-41666-z 
575.	Zhang H, Lu T, Liu S, Yang J, Sun G, Cheng T, et al. Comprehensive understanding of TN5 insertion preference improves transcription regulatory element identification. NAR Genomics and Bioinformatics. 2021 Oct 4;3(4). doi:10.1093/nargab/lqab094 
576.	Kaya-Okur HS, Janssens DH, Henikoff JG, Ahmad K, Henikoff S. Efficient low-cost chromatin profiling with CUT&TAG. Nature Protocols. 2020 Sept 10;15(10):3264–83. doi:10.1038/s41596-020-0373-x 
577.	Meers MP, Tenenbaum D, Henikoff S. Peak calling by sparse enrichment analysis for CUT&RUN chromatin profiling. Epigenetics &amp; Chromatin. 2019 Jul 12;12(1). doi:10.1186/s13072-019-0287-4 
578.	Kaya-Okur HS, Wu SJ, Codomo CA, Pledger ES, Bryson TD, Henikoff JG, et al. CUT&TAG for efficient epigenomic profiling of small samples and single cells. Nature Communications. 2019 Apr 29;10(1). doi:10.1038/s41467-019-09982-5 
579.	Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, et al. Model-based analysis of ChIP-Seq (macs). Genome Biology. 2008 Sept 17;9(9). doi:10.1186/gb-2008-9-9-r137 
580.	Tarbell ED, Liu T. HMMRATAC: A hidden markov modeler for ATAC-Seq. Nucleic Acids Research. 2019 Jun 14;47(16). doi:10.1093/nar/gkz533 
581.	Yang Y, Fear J, Hu J, Haecker I, Zhou L, Renne R, et al. Leveraging biological replicates to improve analysis in CHIP-seq experiments. Computational and Structural Biotechnology Journal. 2014 Jan;9(13). doi:10.5936/csbj.201401002 
582.	Badia-i-Mompel P, Vélez Santiago J, Braunger J, Geiss C, Dimitrov D, Müller-Dott S, et al. DecoupleR: Ensemble of computational methods to infer biological activities from OMICS data. Bioinformatics Advances. 2022 Jan 1;2(1). doi:10.1093/bioadv/vbac016 
583.	Momepel. Decoupler.mt.ulm [Internet]. 2025 [cited 2025 Nov 20]. Available from: https://decoupler.readthedocs.io/en/latest/api/generated/decoupler.mt.ulm.html 
584.	Bentsen M, Goymann P, Schultheis H, Klee K, Petrova A, Wiegandt R, et al. ATAC-seq footprinting unravels kinetics of transcription factor binding during zygotic genome activation. Nature Communications. 2020 Aug 26;11(1). doi:10.1038/s41467-020-18035-1 
585.	Hecker D, Lauber M, Behjati Ardakani F, Ashrafiyan S, Manz Q, Kersting J, et al. Computational tools for inferring transcription factor activity. PROTEOMICS. 2023 Sept 14;23(23–24). doi:10.1002/pmic.202200462 
586.	Li Z, Schulz MH, Look T, Begemann M, Zenke M, Costa IG. Identification of transcription factor binding sites using ATAC-Seq. Genome Biology. 2019 Feb 26;20(1). doi:10.1186/s13059-019-1642-2 
587.	Baek S, Goldstein I, Hager GL. Bivariate genomic footprinting detects changes in transcription factor activity. Cell Reports. 2017 May;19(8):1710–22. doi:10.1016/j.celrep.2017.05.003 
588.	Bailey TL, Johnson J, Grant CE, Noble WS. The meme suite. Nucleic Acids Research. 2015 May 7;43(W1). doi:10.1093/nar/gkv416 
589.	Rodríguez-Jiménez P, Fernández-Messina L, Ovejero-Benito MC, Chicharro P, Vera-Tomé P, Vara A, et al. Growth arrest and DNA damage-inducible proteins (GADD45) in psoriasis. Scientific Reports. 2021 Jul 16;11(1). doi:10.1038/s41598-021-93780-x 
590.	Palomer X, Salvador JM, Griñán‐Ferré C, Barroso E, Pallàs M, Vázquez‐Carrera M. GADD45A: With or without you. Medicinal Research Reviews. 2024 Jan 24;44(4):1375–403. doi:10.1002/med.22015 
591.	Castro-Mondragon JA, Riudavets-Puig R, Rauluseviciute I, Berhanu Lemma R, Turchi L, Blanc-Mathieu R, et al. Jaspar 2022: The 9th release of the open-access database of transcription factor binding profiles. Nucleic Acids Research. 2021 Nov 30;50(D1). doi:10.1093/nar/gkab1113 
592.	Rapisarda V, Malashchuk I, Asamaowei IE, Poterlowicz K, Fessing MY, Sharov AA, et al. P63 transcription factor regulates nuclear shape and expression of nuclear envelope-associated genes in epidermal keratinocytes. Journal of Investigative Dermatology. 2017 Oct;137(10):2157–67. doi:10.1016/j.jid.2017.05.013 
593.	Gdula MR, Poterlowicz K, Mardaryev AN, Sharov AA, Peng Y, Fessing MY, et al. Remodeling of three-dimensional organization of the nucleus during terminal keratinocyte differentiation in the epidermis. Journal of Investigative Dermatology. 2013 Sept;133(9):2191–201. doi:10.1038/jid.2013.66 
594.	Martin CJ, Oser EA, Nagarajan P, Popova LV, Sunkel BD, Stanton BZ, et al. Distinct classes of lamina-associated domains are defined by differential patterns of repressive histone methylation. Genome Research. 2025 Aug 5; doi:10.1101/gr.280380.124 
595.	Primerano A, De Domenico E, Cianfarani F, De Luca N, Floriddia G, Teson M, et al. Histone deacetylase inhibition mitigates fibrosis-driven disease progression in recessive dystrophic epidermolysis bullosa. British Journal of Dermatology. 2024 May 31;191(4):568–79. doi:10.1093/bjd/ljae225 
596.	Gray ZH, Honer MA, Ghatalia P, Shi Y, Whetstine JR. 20 years of histone lysine demethylases: From discovery to the Clinic and Beyond. Cell. 2025 Apr;188(7):1747–83. doi:10.1016/j.cell.2025.02.023 
597.	Eckert RL, Broome A-M, Ruse M, Robinson N, Ryan D, Lee K. S100 proteins in the epidermis. Journal of Investigative Dermatology. 2004 Jul;123(1):23–33. doi:10.1111/j.0022-202x.2004.22719.x 
598.	Abdi W, Romasco A, Alkurdi D, Santacruz E, Okinedo I, Zhang Y, et al. An overview of S100 proteins and their functions in skin homeostasis, interface dermatitis conditions and other skin pathologies. Experimental Dermatology. 2024 Aug;33(8). doi:10.1111/exd.15158 
599.	Vaute O. Functional and physical interaction between the histone methyl transferase SUV39H1 and histone deacetylases. Nucleic Acids Research. 2002 Jan 15;30(2):475–81. doi:10.1093/nar/30.2.475 
600.	Chuang J-Y, Hung J-J. Overexpression of HDAC1 induces cellular senescence by SP1/PP2A/PRB pathway. Biochemical and Biophysical Research Communications. 2011 Apr;407(3):587–92. doi:10.1016/j.bbrc.2011.03.068 
601.	Lee Y, Song MJ, Park JH, Shin MH, Kim M-K, Hwang D, et al. Histone deacetylase 4 reverses cellular senescence via DDIT4 in dermal fibroblasts. Aging. 2022 Jun 9;14(11):4653–72. doi:10.18632/aging.204118 
602.	Warnon C, Bouhjar K, Ninane N, Verhoyen M, Fattaccioli A, Fransolet M, et al. HDAC2 and 7 down-regulation induces senescence in dermal fibroblasts. Aging. 2021 Jul 12;13(14):17978–8005. doi:10.18632/aging.203304 
603.	Willis-Martinez D, Richards HW, Timchenko NA, Medrano EE. Role of HDAC1 in senescence, aging, and cancer. Experimental Gerontology. 2010 Apr;45(4):279–85. doi:10.1016/j.exger.2009.10.001 
604.	Haustead DJ, Stevenson A, Saxena V, Marriage F, Firth M, Silla R, et al. Transcriptome analysis of human ageing in male skin shows mid-life period of variability and central role of NF-ΚB. Scientific Reports. 2016 May 27;6(1). doi:10.1038/srep26846 
605.	Yi Y, Xie H, Xiao X, Wang B, Du R, Liu Y, et al. Ultraviolet a irradiation induces senescence in human dermal fibroblasts by down-regulating DNMT1 via ZEB1. Aging. 2018 Feb 16;10(2):212–28. doi:10.18632/aging.101383 
606.	Sen GL, Reuter JA, Webster DE, Zhu L, Khavari PA. DNMT1 maintains progenitor function in self-renewing somatic tissue. Nature. 2010 Jan 17;463(7280):563–7. doi:10.1038/nature08683 
607.	Beck MA, Fischer H, Grabner LM, Groffics T, Winter M, Tangermann S, et al. DNA hypomethylation leads to cgas‐induced autoinflammation in the epidermis. The EMBO Journal. 2021 Sept 29;40(22). doi:10.15252/embj.2021108234 
608.	Köhler F, Rodríguez-Paredes M. DNA methylation in epidermal differentiation, aging, and cancer. Journal of Investigative Dermatology. 2020 Jan;140(1):38–47. doi:10.1016/j.jid.2019.05.011 
609.	Grönniger E, Max H, Lyko F. Skin rejuvenation by modulation of                    dna                    methylation. Experimental Dermatology. 2024 Oct;33(10). doi:10.1111/exd.70005 
610.	Xie H, Liu Y, Du R, Wang B, Chen M, Zhang Y, et al. Mir-377 induces senescence in human skin fibroblasts by targeting DNA methyltransferase 1. Cell Death &amp; Disease. 2017 Mar 9;8(3). doi:10.1038/cddis.2017.75 
611.	Hirt N, Manchon E, Chen Q, Delaroque C, Corneau A, Hemon P, et al. Systems immunology integrates the complex endotypes of recessive dystrophic epidermolysis bullosa. Nature Communications. 2025 Jan 14;16(1). doi:10.1038/s41467-025-55934-7 
612.	Zhang S, Fei T, Zhang L, Zhang R, Chen F, Ning Y, et al. Smad7 antagonizes transforming growth factor β signaling in the nucleus by interfering with functional Smad-DNA complex formation. Molecular and Cellular Biology. 2007 Jun 1;27(12):4488–99. doi:10.1128/mcb.01636-06 
613.	Rivetti di Val Cervo P, Lena AM, Nicoloso M, Rossi S, Mancini M, Zhou H, et al. P63–microrna feedback in keratinocyte senescence. Proceedings of the National Academy of Sciences. 2012 Jan 6;109(4):1133–8. doi:10.1073/pnas.1112257109 
614.	Ko D, Mun S, Kim M, Nho Y, Lee D, Kang S, et al. A glance into the destiny of transcriptomic activity, embodied by the                     hox                    genes, in neonatal and aging dermal cells. Advanced Biology. 2024 Feb 11;8(4). doi:10.1002/adbi.202300325 
615.	Dudda JC, Perdue N, Bachtanian E, Campbell DJ. Foxp3+ regulatory T cells maintain immune homeostasis in the skin. The Journal of Experimental Medicine. 2008 Jun 23;205(7):1559–65. doi:10.1084/jem.20072594 
616.	Bukowska J, Kopcewicz M, Walendzik K, Gawronska-Kozak B. FOXN1 in skin development, homeostasis and wound healing. International Journal of Molecular Sciences. 2018 Jul 4;19(7):1956. doi:10.3390/ijms19071956 
617.	Pecorari R, Mancini M, Smirnov A, Corleone G, Novelli F, Lena AM, et al. P63 and ZNF148 cooperate to regulate head and neck squamous cell carcinoma. Proceedings of the National Academy of Sciences. 2025 Jul 7;122(28). doi:10.1073/pnas.2500579122 
618.	Xiao Y, Chen X, Chen Z, Dai W, Hu X, Zhang S, et al. Comparative single‐cell transcriptomic analysis across tissues of aging primates reveals specific autologous activation of ZNF281 to mitigate oxidative stress in cornea. Aging Cell. 2024 Sept 10;23(12). doi:10.1111/acel.14319 
619.	Eckert RL, Adhikary G, Young CA, Jans R, Crish JF, Xu W, et al. AP1 transcription factors in epidermal differentiation and skin cancer. Journal of Skin Cancer. 2013;2013:1–9. doi:10.1155/2013/537028 
620.	Patrick R, Naval-Sanchez M, Deshpande N, Huang Y, Zhang J, Chen X, et al. The activity of early-life gene regulatory elements is hijacked in aging through pervasive AP-1-linked chromatin opening. Cell Metabolism. 2024 Aug;36(8). doi:10.1016/j.cmet.2024.06.006 
621.	Zou Z, Long X, Zhao Q, Zheng Y, Song M, Ma S, et al. A single-cell transcriptomic atlas of human skin aging. Developmental Cell. 2021 Feb;56(3). doi:10.1016/j.devcel.2020.11.002 
622.	Mohrin M, Jasper H. Exploring human skin aging at the single-cell level. Developmental Cell. 2021 Feb;56(3):253–4. doi:10.1016/j.devcel.2021.01.006 
623.	Kim DS, Risca VI, Reynolds DL, Chappell J, Rubin AJ, Jung N, et al. The Dynamic, combinatorial cis-regulatory lexicon of epidermal differentiation. Nature Genetics. 2021 Oct 14;53(11):1564–76. doi:10.1038/s41588-021-00947-3 
624.	Wang P, Lv C, Zhang T, Liu J, Yang J, Guan F, et al. Foxq1 regulates senescence-associated inflammation via activation of SIRT1 expression. Cell Death &amp; Disease. 2017 Jul 20;8(7). doi:10.1038/cddis.2017.340 
625.	Bukowska J, Kopcewicz M, Walendzik K, Gawronska-Kozak B. FOXN1 in skin development, homeostasis and wound healing. International Journal of Molecular Sciences. 2018 Jul 4;19(7):1956. doi:10.3390/ijms19071956 
626.	Reis MD, Csomos K, Dias LP, Prodan Z, Szerafin T, Savino W, et al. Decline of FOXN1 gene expression in human thymus correlates with age: Possible epigenetic regulation. Immunity &amp; Ageing. 2015 Oct 29;12(1). doi:10.1186/s12979-015-0045-9 
627.	Paramio JM, Segrelles C, Casanova ML, Jorcano JL. Opposite functions for E2F1 and E2F4 in human epidermal keratinocyte differentiation. Journal of Biological Chemistry. 2000 Dec;275(52):41219–26. doi:10.1074/jbc.m004973200 
628.	Jones SJ, Dicker AJ, Dahler AL, Saunders NA. E2F as a regulator of keratinocyte proliferation: Implications for skin tumor development. Journal of Investigative Dermatology. 1997 Aug;109(2):187–93. doi:10.1111/1523-1747.ep12319308 
629.	Dai X, Segre JA. Transcriptional control of epidermal specification and differentiation. Current Opinion in Genetics &amp; Development. 2004 Oct;14(5):485–91. doi:10.1016/j.gde.2004.07.002 
630.	Gatti V, Fierro C, Annicchiarico‐Petruzzelli M, Melino G, Peschiaroli A. ΔNp63 in squamous cell carcinoma: Defining the oncogenic routes affecting epigenetic landscape and tumour microenvironment. Molecular Oncology. 2019 Mar 22;13(5):981–1001. doi:10.1002/1878-0261.12473 
631.	Sadu Murari LS, Kunkel S, Shetty A, Bents A, Bhandary A, Rivera-Mulia JC. P63: A master regulator at the crossroads between development, Senescence, aging, and cancer. Cells. 2025 Jan 3;14(1):43. doi:10.3390/cells14010043 
632.	Botchkarev VA, Fessing MY, Sharov AA. Deciphering a message from the nucleus: How transcription factors and spatial chromatin interactions orchestrate epidermal differentiation. Journal of Investigative Dermatology. 2023 Jul;143(7):1117–20. doi:10.1016/j.jid.2023.02.013 
633.	Lee CAA, Wu S, Chow YT, Kofman E, Williams V, Riddle M, et al. Accelerated aging and microsatellite instability in recessive dystrophic epidermolysis bullosa–associated cutaneous squamous cell carcinoma. Journal of Investigative Dermatology. 2024 Jul;144(7). doi:10.1016/j.jid.2023.11.025 
634.	Shin J-W, Choi H-R, Nam K-M, Lee H-S, Kim S-A, Joe H-J, et al. The co-expression pattern of P63 and HDAC1: A potential way to disclose stem cells in interfollicular epidermis. International Journal of Molecular Sciences. 2017 Jun 26;18(7):1360. doi:10.3390/ijms18071360 
635.	De Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW, et al. L1 drives IFN in senescent cells and promotes age-associated inflammation. Nature. 2019 Feb 6;566(7742):73–8. doi:10.1038/s41586-018-0784-9 
636.	Dvorkin S, Cambier S, Volkman HE, Stetson DB. New frontiers in the CGAS-sting intracellular DNA-sensing pathway. Immunity. 2024 Apr;57(4):718–30. doi:10.1016/j.immuni.2024.02.019 
637.	Bruno M, Mahgoub M, Macfarlan TS. The arms race between Krab–zinc finger proteins and endogenous retroelements and its impact on mammals. Annual Review of Genetics. 2019 Dec 3;53(1):393–416. doi:10.1146/annurev-genet-112618-043717 
638.	Zuo Z. Quantifying the arms race between line-1 and krab-zinc finger genes through TECookbook. NAR Genomics and Bioinformatics. 2023 Jul 5;5(3). doi:10.1093/nargab/lqad078 
639.	Bulut-Karslioglu A, De La Rosa-Velázquez IA, Ramirez F, Barenboim M, Onishi-Seebacher M, Arand J, et al. SUV39H-dependent h3k9me3 marks intact retrotransposons and silences line elements in mouse embryonic stem cells. Molecular Cell. 2014 Jul;55(2):277–90. doi:10.1016/j.molcel.2014.05.029 
640.	Guler GD, Tindell CA, Pitti R, Wilson C, Nichols K, KaiWai Cheung T, et al. Repression of stress-induced line-1 expression protects cancer cell subpopulations from lethal drug exposure. Cancer Cell. 2017 Aug;32(2). doi:10.1016/j.ccell.2017.07.002 
641.	Liu X, Liu Z, Wu Z, Ren J, Fan Y, Sun L, et al. Resurrection of endogenous retroviruses during aging reinforces senescence. Cell. 2023 Jan;186(2). doi:10.1016/j.cell.2022.12.017 
642.	Hu H, Ji Q, Song M, Ren J, Liu Z, Wang Z, et al. ZKSCAN3 counteracts cellular senescence by stabilizing heterochromatin. Nucleic Acids Research. 2020 May 19;48(11):6001–18. doi:10.1093/nar/gkaa425 
643.	Pal M, Altamirano-Pacheco L, Schauer T, Torres-Padilla M-E. Reorganization of lamina-associated domains in early mouse embryos is regulated by RNA polymerase II activity. Genes &amp; Development. 2023 Oct 1;37(19–20):901–12. doi:10.1101/gad.350799.123 
644.	Peric-Hupkes D, Meuleman W, Pagie L, Bruggeman SWM, Solovei I, Brugman W, et al. Molecular maps of the reorganization of genome-nuclear lamina interactions during differentiation. Molecular Cell. 2010 May;38(4):603–13. doi:10.1016/j.molcel.2010.03.016 
645.	Gdula MR, Poterlowicz K, Mardaryev AN, Sharov AA, Peng Y, Fessing MY, et al. Remodeling of three-dimensional organization of the nucleus during terminal keratinocyte differentiation in the epidermis. Journal of Investigative Dermatology. 2013 Sept;133(9):2191–201. doi:10.1038/jid.2013.66 
646.	Mardaryev AN, Gdula MR, Yarker JL, Emelianov VN, Poterlowicz K, Sharov AA, et al. P63 and BRG1 control developmentally regulated higher-order chromatin remodelling at the epidermal differentiation complex locus in epidermal progenitor cells. Development. 2014 Jan 1;141(1):101–11. doi:10.1242/dev.103200
647.	Williams RRE, Broad S, Sheer D, Ragoussis J. SUBCHROMOSOMAL positioning of the epidermal differentiation complex (EDC) in keratinocyte and lymphoblast interphase nuclei. Experimental Cell Research. 2002 Jan;272(2):163–75. doi:10.1006/excr.2001.5400 
648.	Chang W, Wang Y, Luxton GW, Östlund C, Worman HJ, Gundersen GG. Imbalanced nucleocytoskeletal connections create common polarity defects in progeria and physiological aging. Proceedings of the National Academy of Sciences. 2019 Feb 11;116(9):3578–83. doi:10.1073/pnas.1809683116 
649.	Meqbel BR, Gomes M, Omer A, Gallouzi IE, Horn HF. Lincing senescence and nuclear envelope changes. Cells. 2022 May 30;11(11):1787. doi:10.3390/cells11111787 
650.	Kind J, Pagie L, Ortabozkoyun H, Boyle S, de Vries SS, Janssen H, et al. Single-cell dynamics of genome-nuclear lamina interactions. Cell. 2013 Mar;153(1):178–92. doi:10.1016/j.cell.2013.02.028 
651.	Németh A, Conesa A, Santoyo-Lopez J, Medina I, Montaner D, Péterfia B, et al. Initial genomics of the human nucleolus. PLoS Genetics. 2010 Mar 26;6(3). doi:10.1371/journal.pgen.1000889 
652.	van Koningsbruggen S, Gierliński M, Schofield P, Martin D, Barton GJ, Ariyurek Y, et al. High-resolution whole-genome sequencing reveals that specific chromatin domains from most human chromosomes associate with nucleoli. Molecular Biology of the Cell. 2010 Nov;21(21):3735–48. doi:10.1091/mbc.e10-06-0508 
653.	Bersaglieri C, Kresoja-Rakic J, Gupta S, Bär D, Kuzyakiv R, Panatta M, et al. Genome-wide maps of nucleolus interactions reveal distinct layers of repressive chromatin domains. Nature Communications. 2022 Mar 18;13(1). doi:10.1038/s41467-022-29146-2 
654.	Vertii A, Ou J, Yu J, Yan A, Pagès H, Liu H, et al. Two contrasting classes of nucleolus-associated domains in mouse fibroblast heterochromatin. Genome Research. 2019 Jun 14;29(8):1235–49. doi:10.1101/gr.247072.118 
655.	Bizhanova A, Yan A, Yu J, Zhu LJ, Kaufman PD. Distinct features of nucleolus-associated domains in mouse embryonic stem cells. Chromosoma. 2020 Mar 26;129(2):121–39. doi:10.1007/s00412-020-00734-9 
656.	Shah PP, Keough KC, Gjoni K, Santini GT, Abdill RJ, Wickramasinghe NM, et al. An atlas of lamina-associated chromatin across twelve human cell types reveals an intermediate chromatin subtype. Genome Biology. 2023 Jan 23;24(1). doi:10.1186/s13059-023-02849-5 
657.	Tran JR, Zheng X, Adam SA, Goldman RD, Zheng Y. High quality mapping of chromatin at or near the nuclear lamina from small numbers of cells reveals cell cycle and developmental changes of chromatin at the nuclear periphery. Nucleic Acids Research. 2022 Sept 21;50(20). doi:10.1093/nar/gkac762 
658.	Chen Y, Zhang Y, Wang Y, Zhang L, Brinkman EK, Adam SA, et al. Mapping 3D genome organization relative to nuclear compartments using TSA-Seq as a cytological ruler. Journal of Cell Biology. 2018 Aug 28;217(11):4025–48. doi:10.1083/jcb.201807108 
659.	van Schaik T, Vos M, Peric‐Hupkes D, HN Celie P, van Steensel B. Cell Cycle Dynamics of lamina‐associated DNA. EMBO reports. 2020 Sept 7;21(11). doi:10.15252/embr.202050636 
660.	Gholamalamdari O, van Schaik T, Wang Y, Kumar P, Zhang L, Zhang Y, et al. Major nuclear locales define nuclear genome organization and function beyond A and B compartments. eLife. 2025 Apr 25;13. doi:10.7554/elife.99116.4 
661.	Peng T, Hou Y, Meng H, Cao Y, Wang X, Jia L, et al. Mapping nucleolus-associated chromatin interactions using nucleolus hi-C reveals pattern of heterochromatin interactions. Nature Communications. 2023 Jan 21;14(1). doi:10.1038/s41467-023-36021-1 
662.	Bersaglieri C, Santoro R. Methods for mapping 3D-chromosome architecture around nucleoli. Current Opinion in Cell Biology. 2023 Apr;81:102171. doi:10.1016/j.ceb.2023.102171 
663.	Zeller P, Yeung J, Viñas Gaza H, de Barbanson BA, Bhardwaj V, Florescu M, et al. Single-cell sortchic identifies hierarchical chromatin dynamics during hematopoiesis. Nature Genetics. 2022 Dec 20;55(2):333–45. doi:10.1038/s41588-022-01260-3 
664.	Martin CJ, Nagarajan P, Oser EA, Popova LV, Parthun MR. Distinct classes of lamin-associated domains are defined by differential patterns of repressive histone methylation. 2024 Dec 20; doi:10.1101/2024.12.20.629719 
665.	Gupta S, Bersaglieri C, Bär D, Raingeval M, Schaab L, Santoro R. The nucleolar granular component mediates genome-nucleolus interactions and establishes their repressive chromatin states. Molecular Cell. 2025 Jun;85(11). doi:10.1016/j.molcel.2025.05.004 
666.	Czapiewski R, Schirmer EC. Enhancers on the edge — how the nuclear envelope controls gene regulatory elements. Current Opinion in Genetics &amp; Development. 2024 Aug;87:102234. doi:10.1016/j.gde.2024.102234 
667.	Sen S, Estève P-O, Raman K, Beaulieu J, Chin HG, Feehery GR, et al. Distinct structural and functional heterochromatin partitioning of lamin B1 and lamin B2 revealed using genome-wide nicking enzyme epitope targeted DNA sequencing. Nucleic Acids Research. 2025 Jan 11;53(2). doi:10.1093/nar/gkae1317 
668.	Corman A, Sirozh O, Lafarga V, Fernandez-Capetillo O. Targeting the nucleolus as a therapeutic strategy in human disease. Trends in Biochemical Sciences. 2023 Mar;48(3):274–87. doi:10.1016/j.tibs.2022.09.006 
669.	Gutierrez JI, Tyler JK. A mortality timer based on nucleolar size triggers nucleolar integrity loss and catastrophic genomic instability. Nature Aging. 2024 Nov 25;4(12):1782–93. doi:10.1038/s43587-024-00754-5 
670.	Buchwalter A, Hetzer MW. Nucleolar expansion and elevated protein translation in premature aging. Nature Communications. 2017 Aug 30;8(1). doi:10.1038/s41467-017-00322-z 
671.	Gupta S, Santoro R. Regulation and roles of the nucleolus in embryonic stem cells: From ribosome biogenesis to genome organization. Stem Cell Reports. 2020 Dec;15(6):1206–19. doi:10.1016/j.stemcr.2020.08.012 
672.	Mardaryev AN, Gdula MR, Yarker JL, Emelianov VN, Poterlowicz K, Sharov AA, et al. P63 and BRG1 control developmentally regulated higher-order chromatin remodelling at the epidermal differentiation complex locus in epidermal progenitor cells. Development. 2014 Jan 1;141(1):101–11. doi:10.1242/dev.103200 
673.	Li X, Liu N. Advances in understanding line-1 regulation and function in the human genome. Trends in Genetics. 2025 Jul;41(7):577–89. doi:10.1016/j.tig.2025.04.011 
674.	Camps J, Wangsa D, Falke M, Brown M, Case CM, Erdos MR, et al. Loss of lamin B1 results in prolongation of S phase and decondensation of chromosome territories. The FASEB Journal. 2014 Apr 14;28(8):3423–34. doi:10.1096/fj.14-250456 
675.	Marin HC, Allen C, Simental E, Martin EW, Panning B, Al-Sady B, et al. The nuclear periphery confers repression on H3K9me2-marked genes and transposons to shape cell fate. Nature Cell Biology. 2025 Jul 22;27(8):1311–26. doi:10.1038/s41556-025-01703-z 
676.	Marin HC, Allen C, Simental E, Martin EW, Panning B, Al-Sady B, et al. The nuclear periphery confers repression on H3K9me2-marked genes and transposons to shape cell fate. Nature Cell Biology. 2025 Jul 22;27(8):1311–26. doi:10.1038/s41556-025-01703-z 
677.	Cancado de Faria R, Silva LN, Teodoro-Castro B, McCommis KS, Shashkova EV, Gonzalo S. A noncanonical cGAS–sting pathway drives cellular and organismal aging. Proceedings of the National Academy of Sciences. 2025 Jul 10;122(28). doi:10.1073/pnas.2424666122 
678.	te Molder L, de Pereda JM, Sonnenberg A. Regulation of hemidesmosome dynamics and cell signaling by integrin α6β4. Journal of Cell Science. 2021 Sept 15;134(18). doi:10.1242/jcs.259004 
679.	Wang Y, Kitahata H, Kosumi H, Watanabe M, Fujimura Y, Takashima S, et al. Collagen XVII deficiency alters epidermal patterning. Laboratory Investigation. 2022 Jun;102(6):581–8. doi:10.1038/s41374-022-00738-2 
680.	Chi Y-H, Wang W-P, Hung M-C, Liou G-G, Wang J-Y, Chao P-HG. Deformation of the nucleus by tgfβ1 via the remodeling of nuclear envelope and histone isoforms. Epigenetics &amp; Chromatin. 2022 Jan 4;15(1). doi:10.1186/s13072-021-00434-3 
681.	Rapisarda V, Malashchuk I, Asamaowei IE, Poterlowicz K, Fessing MY, Sharov AA, et al. P63 transcription factor regulates nuclear shape and expression of nuclear envelope-associated genes in epidermal keratinocytes. Journal of Investigative Dermatology. 2017 Oct;137(10):2157–67. doi:10.1016/j.jid.2017.05.013 
682.	Kreienkamp R, Graziano S, Coll-Bonfill N, Bedia-Diaz G, Cybulla E, Vindigni A, et al. A cell-intrinsic interferon-like response links replication stress to cellular aging caused by Progerin. Cell Reports. 2018 Feb;22(8):2006–15. doi:10.1016/j.celrep.2018.01.090 
683.	Fujimura T, Tsukahara K, Moriwaki S, Kitahara T, Sano T, Takema Y. Treatment of human skin with an extract of Fucus vesiculosus changes its thickness and mechanical properties. J Cosmet Sci. 2002;53(1):1-9. 
684.	Chen X, Yao H, Kashif M, Revêchon G, Eriksson M, Hu J, et al. A small-molecule ICMT inhibitor delays senescence of Hutchinson-Gilford progeria syndrome cells. eLife. 2021 Feb 2;10. doi:10.7554/elife.63284 
685.	Primerano A, De Domenico E, Cianfarani F, De Luca N, Floriddia G, Teson M, et al. Histone deacetylase inhibition mitigates fibrosis-driven disease progression in recessive dystrophic epidermolysis bullosa. British Journal of Dermatology. 2024 May 31;191(4):568–79. doi:10.1093/bjd/ljae225 
686.	Balmer P, Hariton WVJ, Sayar BS, Jagannathan V, Galichet A, Leeb T, et al. SUV39H2 epigenetic silencing controls fate conversion of epidermal stem and progenitor cells. Journal of Cell Biology. 2021 Feb 18;220(4). doi:10.1083/jcb.201908178 
687.	Gel B, Díez-Villanueva A, Serra E, Buschbeck M, Peinado MA, Malinverni R. Regioner: An R/bioconductor package for the Association analysis of genomic regions based on permutation tests. Bioinformatics. 2015 Sept 30;32(2):289–91. doi:10.1093/bioinformatics/btv562 
688.	Sheffield NC, Bock C. Lola: Enrichment Analysis for genomic region sets and regulatory elements in R and bioconductor. Bioinformatics. 2015 Oct 27;32(4):587–9. doi:10.1093/bioinformatics/btv612 
689.	Qin Q, Fan J, Zheng R, Wan C, Mei S, Wu Q, et al. Lisa: Inferring transcriptional regulators through integrative modeling of Public Chromatin Accessibility and ChIP-Seq Data. Genome Biology. 2020 Feb 7;21(1). doi:10.1186/s13059-020-1934-6 
690.	Horvath S. DNA methylation age of human tissues and cell types. Genome Biology. 2013 Dec 10;14(10). doi:10.1186/gb-2013-14-10-r115 
691.	Wang T-C, Dollahon CR, Mishra S, Patel H, Abolghasemzade S, Singh I, et al. Extreme wrinkling of the nuclear lamina is a morphological marker of cancer. npj Precision Oncology. 2024 Dec 2;8(1). doi:10.1038/s41698-024-00775-8 

